<p><h3>2024</h3></p>
<ul><li>Zhang K, Zemke NR, Armand EJ, <B>Ren B</B>. A fast, scalable and versatile tool for analysis of single-cell omics data. Nat Methods. 2024 Jan 8. doi: 10.1038/s41592-023-02139-9. Online ahead of print
</li></ul>
<p><h3>2023</h3></p>
<ul><li>BB Jia, A Jussila, C Kern, Q Zhu, <B>B Ren</B>. A spatial genome aligner for resolving chromatin architectures from multiplexed DNA FISH. Nature biotechnology, 2023 Jan 2. doi: 10.1038/s41587-022-01568-9. Online ahead of print.
</li></ul>
<ul><li>Kirkland NJ, Skalak SH, Whitehead AJ, Hocker JD, Beri P, Vogler G, Hum B, Wang M, Lakatta EG, <B>Ren B</B>, Bodmer R, Engler AJ. Age-dependent Lamin changes induce cardiac dysfunction via dysregulation of cardiac transcriptional programs. Nat Aging. 2023 Jan;3(1):17-33. 
</li></ul>
<ul><li>Qiu H, Makarov V, Bolzenius JK, Halstead A, Parker Y, Wang A, Iyer GV, Wise H, Kim D, Thayaparan V, Lindner DJ, Haber GP, Ting AH, <B>Ren B</B>, Chan TA, Arora V, Solit DB, Lee BH. KDM6A Loss Triggers an Epigenetic Switch That Disrupts Urothelial Differentiation and Drives Cell Proliferation in Bladder Cancer. Cancer Res. 2023 Mar 15;83(6):814-829. doi: 10.1158/0008-5472.CAN-22-1444.
</li></ul>
<ul><li>Rozowsky J, Gao J, Borsari B, Yang YT, Galeev T, Gürsoy G, Epstein CB, Xiong K, Xu J, Li T, Liu J, Yu K, Berthel A, Chen Z, Navarro F, Sun MS, Wright J, Chang J, Cameron CJF, Shoresh N, Gaskell E, Drenkow J, Adrian J, Aganezov S, Aguet F, Balderrama-Gutierrez G, Banskota S, Corona GB, Chee S, Chhetri SB, Cortez Martins GC, Danyko C, Davis CA, Farid D, Farrell NP, Gabdank I, Gofin Y, Gorkin DU, Gu M, Hecht V, Hitz BC, Issner R, Jiang Y, Kirsche M, Kong X, Lam BR, Li S, Li B, Li X, Lin KZ, Luo R, Mackiewicz M, Meng R, Moore JE, Mudge J, Nelson N, Nusbaum C, Popov I, Pratt HE, Qiu Y, Ramakrishnan S, Raymond J, Salichos L, Scavelli A, Schreiber JM, Sedlazeck FJ, See LH, Sherman RM, Shi X, Shi M, Sloan CA, Strattan JS, Tan Z, Tanaka FY, Vlasova A, Wang J, Werner J, Williams B, Xu M, Yan C, Yu L, Zaleski C, Zhang J, Ardlie K, Cherry JM, Mendenhall EM, Noble WS, Weng Z, Levine ME, Dobin A, Wold B, Mortazavi A, <B>Ren B</B>, Gillis J, Myers RM, Snyder MP, Choudhary J, Milosavljevic A, Schatz MC, Bernstein BE, Guigó R, Gingeras TR, Gerstein M. The EN-TEx resource of multi-tissue personal epigenomes & variant-impact models. Cell. 2023 Mar 30;186(7):1493-1511.e40. doi: 10.1016/j.cell.2023.02.018. PMID: 37001506
</li></ul>
<ul><li>Rajderkar S, Barozzi I, Zhu Y, Hu R, Zhang Y, Li B, Alcaina Caro A, Fukuda-Yuzawa Y, Kelman G, Akeza A, Blow MJ, Pham Q, Harrington AN, Godoy J, Meky EM, von Maydell K, Hunter RD, Akiyama JA, Novak CS, Plajzer-Frick I, Afzal V, Tran S, Lopez-Rios J, Talkowski ME, Lloyd KCK, <B>Ren B</B>, Dickel DE, Visel A, Pennacchio LA. Topologically associating domain boundaries are required for normal genome function. Commun Biol. 2023 Apr 20;6(1):435. doi: 10.1038/s42003-023-04819-w. PMID: 37081156
</li></ul>
<ul><li>Zhou P, VanDusen NJ, Zhang Y, Cao Y, Sethi I, Hu R, Zhang S, Wang G, Ye L, Mazumdar N, Chen J, Zhang X, Guo Y, Li B, Ma Q, Lee JY, Gu W, Yuan GC, <B>Ren B</B>, Chen K, Pu WT. Dynamic changes in P300 enhancers and enhancer-promoter contacts control mouse cardiomyocyte maturation. Dev Cell. 2023 Apr 11:S1534-5807(23)00136-3. doi: 10.1016/j.devcel.2023.03.020. Online ahead of print. PMID: 37071996
</li></ul>
<ul><li>Raviram R, Raman A, Preissl S, Ning J, Wu S, Koga T, Zhang K, Brennan CW, Zhu C, Luebeck J, Van Deynze K, Han JY, Hou X, Ye Z, Mischel AK, Li YE, Fang R, Baback T, Mugford J, Han CZ, Glass CK, Barr CL, Mischel PS, Bafna V, Escoubet L, <B>Ren B</B>*, Chen CC*. Integrated analysis of single-cell chromatin state and transcriptome identified common vulnerability despite glioblastoma heterogeneity. Proc Natl Acad Sci U S A. 2023 May 16;120(20):e2210991120. doi: 10.1073/pnas.2210991120. Epub 2023 May 8. PMID: 37155843.  * Co-correspondence.
</li></ul>
<ul><li>Fixsen BR, Han CZ, Zhou Y, Spann NJ, Saisan P, Shen Z, Balak C, Sakai M, Cobo I, Holtman IR, Warden AS, Ramirez G, Collier JG, Pasillas MP, Yu M, Hu R, Li B, Belhocine S, Gosselin D, Coufal NG, <B>Ren B</B>, Glass CK. SALL1 enforces microglia-specific DNA binding and function of SMADs to establish microglia identity. Nat Immunol 24, 1188–1199 (2023).
</li></ul>
<ul><li>Xie F, Armand EJ, Yao Z, Liu H, Bartlett A, Behrens MM, Li YE, Lucero JD, Luo C, Nery JR, Pinto-Duarte A, Poirion OB, Preissl S, Rivkin AC, Tasic B, Zeng H, <B>Ren B</B>, Ecker JR, Mukamel EA. Robust enhancer-gene regulation identified by single-cell transcriptomes and epigenomes. Cell Genom. 2023 Jun 19;3(7):100342.
</li></ul>
<ul><li>Xie Y, Zhu C, Wang Z, Tastemel M, Chang L, Li YE, <B>Ren B</B>. Droplet-based single-cell joint profiling of histone modifications and transcriptomes. Nat Struct Mol Biol. 2023 Aug 10.
</li></ul>
<ul><li>Lee L, Yu H, Jia BB, Jussila A, Zhu C, Chen J, Xie L, Hafner A, Mishra S, Wang DD, Strambio-De-Castillia C, Boettiger A, <B>Ren B</B>, Li Y, Hu M. SnapFISH: a computational pipeline to identify chromatin loops from multiplexed DNA FISH data. Nat Commun. 2023 Aug 12;14(1):4873.
</li></ul>
<ul><li>Lee L, Yu M, Li X, Zhu C, Zhang Y, Yu H, Chen Z, Mishra S, <B>Ren B</B>, Li Y, Hu M. SnapHiC-D: a computational pipeline to identify differential chromatin contacts from single-cell Hi-C data. Brief Bioinform. 2023 Aug 30:bbad315.
</li></ul>
<ul><li>Li YE, Preissl S, Miller M, Johnson ND, Wang Z, Jiao H, Zhu C, Wang Z, Xie Y, Poirion O, Kern C, Pinto-Duarte A, Tian W, Siletti K, Emerson N, Osteen J, Lucero J, Lin L, Yang Q, Zhu Q, Zemke N, Espinoza S, Yanny AM, Nyhus J, Dee N, Casper T, Shapovalova N, Hirschstein D, Hodge RD, Linnarsson S, Bakken T, Levi B, Keene CD, Shang J, Lein E, Wang A, Behrens MM, Ecker JR, <B>Ren B</B>. A comparative atlas of single-cell chromatin accessibility in the human brain. Science. 2023 Oct 13;382(6667):eadf7044. doi: 10.1126/science.adf7044. 
</li></ul>
<ul><li>Tian W, Zhou J, Bartlett A, Zeng Q, Liu H, Castanon RG, Kenworthy M, Altshul J, Valadon C, Aldridge A, Nery JR, Chen H, Xu J, Johnson ND, Lucero J, Osteen JK, Emerson N, Rink J, Lee J, Li YE, Siletti K, Liem M, Claffey N, O'Connor C, Yanny AM, Nyhus J, Dee N, Casper T, Shapovalova N, Hirschstein D, Ding SL, Hodge R, Levi BP, Keene CD, Linnarsson S, Lein E, <B>Ren B</B>, Behrens MM, Ecker JR. Single-cell DNA methylation and 3D genome architecture in the human brain. Science. 2023 Oct 13;382(6667):eadf5357. doi: 10.1126/science.adf5357.
</li></ul>
<ul><li>Calandrelli R, Wen X, Charles Richard JL, Luo Z, Nguyen TC, Chen CJ, Qi Z, Xue S, Chen W, Yan Z, Wu W, Zaleta-Rivera K, Hu R, Yu M, Wang Y, Li W, Ma J, <B>Ren B</B>, Zhong S. Genome-wide analysis of the interplay between chromatin-associated RNA and 3D genome organization in human cells. Nat Commun. 2023 Oct 16;14(1):6519. doi: 10.1038/s41467-023-42274-7.
</li></ul>
<ul><li>Whitehead AJ, Atcha H, Hocker JD, <B>Ren B</B>, Engler AJ. AP-1 signaling modulates cardiac fibroblast stress responses. J Cell Sci. 2023 Nov 23:jcs.261152. doi: 10.1242/jcs.261152.
</li></ul>
<ul><li>Zu S, Li YE, Wang K, Armand EJ, Mamde S, Amaral ML, Wang Y, Chu A, Xie Y, Miller M, Xu J, Wang Z, Zhang K, Jia BB, Hou X, Li B, Kuan S, Liu H, Zhou J, Pinto-Duarte A, Lucero J, Osteen J, Nunn M, Smith KA, Tasic B, Yao Z, Zeng H, Wang Z, Shang J, Behrens MM, Ecker JR, Wang A, Preissl S, <B>Ren B</B>*, Single-cell analysis of chromatin accessibility in the adult mouse brain. Nature, Dec 14, 2023, https://doi.org/10.1038/s41586-023-06824-9
</li></ul>
<ul><li>Zemke NR, Armand EJ, Wang W, Lee S, Zhou J, Li YE, Liu H, Tian W, Nery JR, Castanon RG, Bartlett A, Osteen JK, Li D, Zhuo X, Xu V, Chang L, Dong K, Indralingam H, Rink JA, Xie Y, Miller M, Krienen FM, Zhang Q, Taskin N, Ting J, Feng G, McCarroll SA, Callaway EM, Wang T, Lein ES, Behrens MM,  Ecker JR*, <B>Ren B</B>*, Conserved and divergent gene regulatory programs of the mammalian neocortex. Nature, Dec 14, 2023, https://doi.org/10.1038/s41586-023-06819-6
</li></ul>
<ul><li>Liu H, Zeng Q, Zhou J, Bartlett A, Wang B, Berube P, Tian W, Kenworthy M, Altshul J, Nery JR, Chen H, Castanon RG, Zu S, Li YE, Lucero J, Osteen JK, Pinto-Duarte A, Lee J, Rink J, Cho S, Nora Emerson N, Nunn M, O’Connor C, Wu Z, Stoica I, Yao Z, Smith KA, Tasic B, Luo C, Dixon JR, Zeng H, <B>Ren B</B>, Behrens MM Ecker JR, Single-cell DNA Methylome and 3D Multi-omic Atlas of the Adult Mouse Brain. Nature, Dec 14, 2023
</li></ul>
<p><h3>2022</h3></p>
<ul><li>Whitehead AJ, Hocker JD, <B>Ren B</B>, Engler AJ. Improved epicardial cardiac fibroblast generation from iPSCs. J Mol Cell Cardiol. 2022 Mar;164:58-68. 
</li></ul>
<ul><li>Luo C, Liu H, Xie F, Armand EJ, Siletti K, Bakken TE, Fang R, Doyle WI, Stuart T, Hodge RD, Hu L, Wang BA, Zhang Z, Preissl S, Lee DS, Zhou J, Niu SY, Castanon R, Bartlett A, Rivkin A, Wang X, Lucero J, Nery JR, Davis DA, Mash DC, Satija R, Dixon JR, Linnarsson S, Lein E, Behrens MM, <B>Ren B</B>, Mukamel EA, Ecker JR. Single nucleus multi-omics identifies human cortical cell regulatory genome diversity. Cell Genom. 2022 Mar 9;2(3):100106. 
</li></ul>
<ul><li>Meng Z, Li FL, Fang C, Yeoman B, Qiu Y, Wang Y, Cai X, Lin KC, Yang D, Luo M, Fu V, Ma X, Diao Y, Giancotti FG, <B>Ren B</B>, Engler AJ, Guan KL. The Hippo pathway mediates Semaphorin signaling. Sci Adv. 2022 May 27;8(21):eabl9806. 
</li></ul>
<ul><li>Destici, E., Zhu, F., Tran, S., Preissl,S., Farah,E.N., Zhang, Y., Hou, X., Poiron, O.B., Lee A.Y., Grinstein, J.D., Bloomekatz, J., Kim, H.S., Hu, R., Evans, S.M., <B>Ren, B</B>., Benner C. & Chi, N.C., Human-gained heart enhancers are associated with species-specific cardiac attributes. Nat Cardiovasc Res 1, 830–843.
</li></ul>
<ul><li>Zhang Y, Amaral ML, Zhu C, Grieco SF, Hou X, Lin L, Buchanan J, Tong L, Preissl S, Xu X, <B>Ren B</B>. Single-cell epigenome analysis reveals age-associated decay of heterochromatin domains in excitatory neurons in the mouse brain. Cell Res. 32, pages 1008–1021, 2022 Oct 7. 
</li></ul>
<ul><li>Chen PB, Fiaux PC, Zhang K, Li B, Kubo N, Jiang S, Hu R, Rooholfada E, Wu S, Wang M, Wang W, McVicker G, Mischel PS, <B>Ren B</B>. Systematic discovery and functional dissection of enhancers needed for cancer cell fitness and proliferation. Cell Rep. 2022 Nov 8;41(6):111630. 
</li></ul>
<ul><li>Benaglio P, Zhu H, Okino ML, Yan J, Elgamal R, Nariai N, Beebe E, Korgaonkar K, Qiu Y, Donovan MKR, Chiou J, Wang G, Newsome J, Kaur J, Miller M, Preissl S, Corban S, Aylward A, Taipale J, <B>Ren B</B>, Frazer KA, Sander M, Gaulton KJ.  Type 1 diabetes risk genes mediate pancreatic beta cell survival in response to proinflammatory cytokines. Cell Genomics. 2022 Nov 11. 100214.
</li></ul>
<p><h3>2021</h3></p>
<ul><li>Yan J*, Qiu Y, dos Santos AMR, Yin Y, Li YE, Vinckier N, Nariai N, Benaglio P, Raman A, Li X, Fan S, Chiou J, Chen F, Frazer KA, Gaulton KJ, Sander M, Taipale J* and <B>Ren B</B>*.  Systematic Analysis of Transcription Factor Binding to Noncoding Variants in the Human Genome, Nature 2021 Jan 27. (*co-corresponding author)
</li></ul>
<ul><li>Kubo N, Ishii H, Xiong H, Bianco S, Meitinger F, Hu R, Hocker JD, Conte M, Gorkin D, Yu M, Li B, Dixon JR, Hu M, Nicodemi M, Zhao H, and <B>Ren B</B>. Promoter-proximal CTCF-binding promotes Distal-enhancer dependent gene activation.  NSMB 2021 Jan 4.
</li></ul>
<ul><li>Geusz RJ, Wang A, Chiou J, Lancman JJ, Wetton N, Kefalopoulou S, Wang J, Qui Y, Yan J, Aylward A, <B>Ren B</B>, Dong PDS, Gaulton KJ, Sander M. Pancreatic progenitor epigenome maps prioritize type 2 diabetes risk genes with roles in development. Elife. 2021 Feb 5;10:e59067. doi: 10.7554/eLife.59067. PMID: 33544077
</li></ul>
<ul><li>Agarwal S*, Bonefas KM, Garay PM, Brookes E, Murata-Nakamura Y, Porter RS, Macfarlan TS, <B>Ren B</B>*, Iwase S*. KDM1A maintains genome-wide homeostasis of transcriptional enhancers. Genome Res. 2021 Jan 7;31(2):186-97. doi: 10.1101/gr.234559.118. Online ahead of print. PMID: 33414108 (*co-corresponding author)
</li></ul>
<ul><li>Zhou Q, Yu M, Tirado-Magallanes R, Li B, Kong L, Guo M, Tan ZH, Lee S, Chai L, Numata A, Benoukraf T, Fullwood MJ, Osato M, <B>Ren B</B>, Tenen DG. ZNF143 mediates CTCF-bound promoter-enhancer loops required for murine hematopoietic stem and progenitor cell function. Nat Commun. 2021 Jan 4;12(1):43. doi: 10.1038/s41467-020-20282-1. PMID: 33397967
</li></ul>
<ul><li>Zhu C, Zhang Y, Li YE, Lucero J, Behrens MM and <B>Ren B</B>. Joint profiling of histone modifications and transcriptome in single cells from mouse cortex and hippocampus.  Nature Methods 2021 Feb 15. doi: 10.1038/s41592-021-01060-3. Online ahead of print.
</li></ul>
<ul><li>Fang R., Preissl S., Hou X., Lucero J, Wang X, Motamedi A, Shiau AK, Mukamel EA, Zhang Y, Behrens MM, Ecker J, and <B>Ren B</B>. SnapATAC: A Comprehensive Analysis Package for Single Cell ATAC-seq. Nat Commun. 2021 Feb 26;12(1):1337. doi: 10.1038/s41467-021-21583-9.
</li></ul>
<ul><li>Guo Q, Kim AD, Li B, Ransick A, Bugacov H, Chen X, Lindström NO, Brown A, Oxburgh L, <B>Ren B</B>, McMahon AP. A β-catenin-driven switch in TCF/LEF transcription factor binding to DNA target sites promotes commitment of mammalian nephron progenitor cells. Elife. 2021 Feb 15;10:e64444. doi: 10.7554/eLife.64444. Online ahead of print. PMID: 33587034
</li></ul>
<ul><li>Gao C, Liu J, Kriebel AR, Preissl S, Luo C, Castanon R, Sandoval J, Rivkin A, Nery JR, Behrens MM, Ecker JR, <B>Ren B</B>, Welch JD. Iterative single-cell multi-omic integration using online learning. Nat Biotechnol. 2021 Apr 19. doi: 10.1038/s41587-021-00867-x.
</li></ul>
<ul><li>Hocker JD, Poirion OB, Zhu F, Buchanan J, Zhang K, Chiou J, Wang TM, Zhang Q, Hou X, Li YE, Zhang Y, Farah EN, Wang A, McCulloch AD, Gaulton KJ, <B>Ren B</B>*, Chi NC*, Preissl S*. Cardiac cell type-specific gene regulatory programs and disease risk association. Sci Adv. 2021 May 14;7(20):eabf1444. (*co-corresponding author)
</li></ul>
<ul><li>Huang H, Zhu Q, Jussila A, Han Y, Bintu B, Kern C, Conte M, Zhang Y, Bianco S, Chiariello AM, Yu M, Hu R, Tastemel M, Juric I, Hu M, Nicodemi M, Zhuang X, <B>Ren B</B>. CTCF mediates dosage- and sequence-context-dependent transcriptional insulation by forming local chromatin domains. Nat Genet. 2021 May 17. doi: 10.1038/s41588-021-00863-6.
</li></ul>
<ul><li>Yu M, Abnousi A, Zhang Y, Li G, Lee L, Chen Z, Fang R, Lagler TM, Yang Y, Wen J, Sun Q, Li Y, <B>Ren B</B>*, Hu M*. SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data. Nat Methods. 2021 Aug 26. Online ahead of print. (*co-corresponding author)
</li></ul>
<ul><li>Bruand M, Barras D, Mina M, Ghisoni E, Morotti M, Lanitis E, Fahr N, Desbuisson M, Grimm A, Zhang H, Chong C, Dagher J, Chee S, Tsianou T, Dorier J, Stevenson BJ, Iseli C, Ronet C, Bobisse S, Genolet R, Walton J, Bassani-Sternberg M, Kandalaft LE, <B>Ren B</B>, McNeish I, Swisher E, Harari A, Delorenzi M, Ciriello G, Irving M, Rusakiewicz S, Foukas PG, Martinon F, Dangaj Laniti D, Coukos G. Cell-autonomous inflammation of BRCA1-deficient ovarian cancers drives both tumor-intrinsic immunoreactivity and immune resistance via STING. Cell Rep. 2021 Jul 20;36(3):109412.
</li></ul>
<ul><li>Ren X, Wang M, Li B, Jamieson K, Zheng L, Jones IR, Li B, Takagi MA, Lee J, Maliskova L, Tam TW, Yu M, Hu R, Lee L, Abnousi A, Li G, Li Y, Hu M, <B>Ren B</B>, Wang W, Shen Y. Parallel characterization of cis-regulatory elements for multiple genes using CRISPRpath. Sci Adv. 2021 Sep 17;7(38):eabi4360. doi: 10.1126/sciadv.abi4360. Epub 2021 Sep 15.
</li></ul>
<ul><li>BRAIN Initiative Cell Census Network (BICCN). A multimodal cell census and atlas of the mammalian primary motor cortex. Nature. 2021 Oct;598(7879):86-102. doi: 10.1038/s41586-021-03950-0. Epub 2021 Oct 6. 
</li></ul>
<ul><li>Li YE, Preissl S, Hou X, Zhang Z, Zhang K, Qiu Y, Poirion OB, Li B, Chiou J, Liu H, Pinto-Duarte A, Kubo N, Yang X, Fang R, Wang X, Han JY, Lucero J, Yan Y, Miller M, Kuan S, Gorkin D, Gaulton KJ, Shen Y, Nunn M, Mukamel EA, Behrens MM, Ecker JR, <B>Ren B</B>. An atlas of gene regulatory elements in adult mouse cerebrum. Nature. 2021 Oct;598(7879):129-136. doi: 10.1038/s41586-021-03604-1. Epub 2021 Oct 6.
</li></ul>
<ul><li>Yao Z, Liu H, Xie F, Fischer S, Adkins RS, Aldridge AI, Ament SA, Bartlett A, Behrens MM, Van den Berge K, Bertagnolli D, de Bézieux HR, Biancalani T, Booeshaghi AS, Bravo HC, Casper T, Colantuoni C, Crabtree J, Creasy H, Crichton K, Crow M, Dee N, Dougherty EL, Doyle WI, Dudoit S, Fang R, Felix V, Fong O, Giglio M, Goldy J, Hawrylycz M, Herb BR, Hertzano R, Hou X, Hu Q, Kancherla J, Kroll M, Lathia K, Li YE, Lucero JD, Luo C, Mahurkar A, McMillen D, Nadaf NM, Nery JR, Nguyen TN, Niu SY, Ntranos V, Orvis J, Osteen JK, Pham T, Pinto-Duarte A, Poirion O, Preissl S, Purdom E, Rimorin C, Risso D, Rivkin AC, Smith K, Street K, Sulc J, Svensson V, Tieu M, Torkelson A, Tung H, Vaishnav ED, Vanderburg CR, van Velthoven C, Wang X, White OR, Huang ZJ, Kharchenko PV, Pachter L, Ngai J, Regev A, Tasic B, Welch JD, Gillis J, Macosko EZ, <B>Ren B</B>, Ecker JR, Zeng H, Mukamel EA. A transcriptomic and epigenomic cell atlas of the mouse primary motor cortex. Nature. 2021 Oct;598(7879):103-110. doi: 10.1038/s41586-021-03500-8. Epub 2021 Oct 6.
</li></ul>
<ul><li>Bakken TE, Jorstad NL, Hu Q, Lake BB, Tian W, Kalmbach BE, Crow M, Hodge RD, Krienen FM, Sorensen SA, Eggermont J, Yao Z, Aevermann BD, Aldridge AI, Bartlett A, Bertagnolli D, Casper T, Castanon RG, Crichton K, Daigle TL, Dalley R, Dee N, Dembrow N, Diep D, Ding SL, Dong W, Fang R, Fischer S, Goldman M, Goldy J, Graybuck LT, Herb BR, Hou X, Kancherla J, Kroll M, Lathia K, van Lew B, Li YE, Liu CS, Liu H, Lucero JD, Mahurkar A, McMillen D, Miller JA, Moussa M, Nery JR, Nicovich PR, Niu SY, Orvis J, Osteen JK, Owen S, Palmer CR, Pham T, Plongthongkum N, Poirion O, Reed NM, Rimorin C, Rivkin A, Romanow WJ, Sedeño-Cortés AE, Siletti K, Somasundaram S, Sulc J, Tieu M, Torkelson A, Tung H, Wang X, Xie F, Yanny AM, Zhang R, Ament SA, Behrens MM, Bravo HC, Chun J, Dobin A, Gillis J, Hertzano R, Hof PR, Höllt T, Horwitz GD, Keene CD, Kharchenko PV, Ko AL, Lelieveldt BP, Luo C, Mukamel EA, Pinto-Duarte A, Preissl S, Regev A, <B>Ren B</B>, Scheuermann RH, Smith K, Spain WJ, White OR, Koch C, Hawrylycz M, Tasic B, Macosko EZ, McCarroll SA, Ting JT, Zeng H, Zhang K, Feng G, Ecker JR, Linnarsson S, Lein ES. Comparative cellular analysis of motor cortex in human, marmoset and mouse. Nature. 2021 Oct;598(7879):111-119. doi: 10.1038/s41586-021-03465-8. Epub 2021 Oct 6.
</li></ul>
<ul><li>Liu H, Zhou J, Tian W, Luo C, Bartlett A, Aldridge A, Lucero J, Osteen JK, Nery JR, Chen H, Rivkin A, Castanon RG, Clock B, Li YE, Hou X, Poirion OB, Preissl S, Pinto-Duarte A, O'Connor C, Boggeman L, Fitzpatrick C, Nunn M, Mukamel EA, Zhang Z, Callaway EM, <B>Ren B</B>, Dixon JR, Behrens MM, Ecker JR. DNA methylation atlas of the mouse brain at single-cell resolution. Nature. 2021 Oct;598(7879):120-128. doi: 10.1038/s41586-020-03182-8. Epub 2021 Oct 6.
</li></ul>
<ul><li>Zhang K, Hocker JD, Miller M, Hou X, Chiou J, Poirion OB, Qiu Y, Li YE, Gaulton KJ, Wang A, Preissl S, <B>Ren B</B>. A single-cell atlas of chromatin accessibility in the human genome.  Cell. 2021 Nov 8:S0092-8674(21)01279-4.
</li></ul>
<ul><li>Geusz RJ, Wang A, Lam DK, Vinckier NK, Alysandratos KD, Roberts DA, Wang J, Kefalopoulou S, Ramirez A, Qiu Y, Chiou J, Gaulton KJ, <B>Ren B</B>, Kotton DN, Sander M. Sequence logic at enhancers governs a dual mechanism of endodermal organ fate induction by FOXA pioneer factors. Nat Commun. 2021 Nov 17;12(1):6636.
</li></ul>
<p><h3>2020</h3></p>
<ul><li>Pugacheva EM, Kubo N, Loukinov D, Tajmul M, Kang S, Kovalchuk AL, Strunnikov AV, Zentner GE, <B>Ren B</B>, Lobanenkov VV. CTCF mediates chromatin looping via N-terminal domain-dependent cohesin retention. Proc Natl Acad Sci U S A. 2020 Jan 14. [Epub ahead of print]
</li></ul>
<ul><li>Arvanitis M, Tampakakis E, Zhang Y, Wang W, Auton A; 23andMe Research Team, Dutta D, Glavaris S, Keramati A, Chatterjee N, Chi NC, <B>Ren B</B>, Post WS, Battle A. Genome-wide association and multi-omic analyses reveal ACTN2 as a gene linked to heart failure. Nat Commun. 2020; 11: 1122.
</li></ul>
<ul><li>Gorkin DU, Barozzi I, Zhao Y, Zhang Y, Huang H, Lee AY, Li B, Chiou J, Wildberg A, Ding B, Zhang B, Wang M, Strattan JS, Davidson JM, Qiu Y, Afzal V, Akiyama JA, Plajzer-Frick I, Novak CS, Kato M, Garvin TH, Pham QT, Harrington AN, Mannion BJ, Lee EA, Fukuda-Yuzawa Y, He Y, Preissl S, Chee S, Han JY, Williams BA, Trout D, Amrhein H, Yang H, Cherry JM, Wang W, Gaulton K, Ecker JR, Shen Y, Dickel DE, Visel A, Pennacchio LA, <B>Ren B</B>, An atlas of dynamic chromatin landscapes in mouse fetal development. Nature 583(7818):744-751
</li></ul>
<ul><li>ENCODE Project Consortium, Moore JE, Purcaro MJ, Pratt HE, Epstein CB, Shoresh N, Adrian J, Kawli T, Davis CA, Dobin A, Kaul R, Halow J, Van Nostrand EL, Freese P, Gorkin DU, Shen Y, He Y, Mackiewicz M, Pauli-Behn F, Williams BA, Mortazavi A, Keller CA, Zhang XO, Elhajjajy SI, Huey J, Dickel DE, Snetkova V, Wei X, Wang X, Rivera-Mulia JC, Rozowsky J, Zhang J, Chhetri SB, Zhang J, Victorsen A, White KP, Visel A, Yeo GW, Burge CB, Lécuyer E, Gilbert DM, Dekker J, Rinn J, Mendenhall EM, Ecker JR, Kellis M, Klein RJ, Noble WS, Kundaje A, Guigó R, Farnham PJ, Cherry JM, Myers RM, <B>Ren B</B>, Graveley BR, Gerstein MB, Pennacchio LA, Snyder MP, Bernstein BE, Wold B, Hardison RC, Gingeras TR, Stamatoyannopoulos JA, Weng Z. Expanded Encyclopedias of DNA Elements in the Human and Mouse Genomes. Nature 583(7818):699-710. (I am one of the co-corresponding authors)
</li></ul>
<ul><li>He Y, Hariharan M, Gorkin DU, Dickel DE, Luo C, Castanon RG, Nery JR, Lee AY, Zhao Y, Huang H, Williams BA, Trout D, Amrhein H, Fang R, Chen H, Li B, Visel A, Pennacchio LA, <B>Ren B</B>, Ecker JR. Spatiotemporal DNA methylome dynamics of the developing mouse fetus. Nature 583(7818):752-759.
</li></ul>
<ul><li>He P, Williams BA, Trout D, Marinov GK, Amrhein H, Berghella L, Goh ST, Plajzer-Frick I, Afzal V, Pennacchio LA, Dickel DE, Visel A, <B>Ren B</B>, Hardison RC, Zhang Y, Wold BJ. The changing mouse embryo transcriptome at whole tissue and single-cell resolution. Nature. 583(7818):760-767. 
</li></ul>
<ul><li>You Q, Cheng AY, Gu X, Harada BT, Yu M, Wu T, <B>Ren B</B>*, Ouyang Z*, He C*. Direct DNA crosslinking with CAP-C uncovers transcription-dependent chromatin organization at high resolution. Nature Biotechnologies, 2020 Aug 24. doi: 10.1038/s41587-020-0643-8. Online ahead of print. (*co-corresponding author)
</li></ul>
<ul><li>Gally F, Sasse SK, Kurche J, Gruca MA, Cardwell JH, Okamoto T, Chu HW, Hou X, Poirion O, Buchanan J, Preissl S, <B>Ren B</B>, Colgan SP, Dowell RD, Yang IV, Schwartz DA, Gerber AN. The MUC5B-associated variant, rs35705950, resides within an enhancer subject to lineage- and disease-dependent epigenetic remodeling. JCI Insight. 2020 Dec 15:144294. 
</li></ul>
<ul><li>Shoshani O, Brunner SF, Yaeger R, Ly P, Nechemia-Arbely, Kim DH, Fang R, Castillon GA, Yu M, Li JSZ, Sun Y, Ellisman MH, <B>Ren B</B>, Campbell PJ & Cleveland DW. Chromothripsis drives the evolution of gene amplification in cancer. Nature, Published online Dec 23, 2020
</li></ul>
<ul><li>Crowley C, Yang Y, Qiu Y, Hu B, Abnousi A, Lipiński J, Plewczyński D, Wu D, Won H, <B>Ren B</B>, Hu M, Li Y. FIREcaller: Detecting frequently interacting regions from Hi-C data. Comput Struct Biotechnol J. 2020 Dec 29;19:355-362. doi: 10.1016/j.csbj.2020.12.026. eCollection 2021. PMID: 33489005
</li></ul>
<p><h3>2019</h3></p>
<ul><li>Larsson AJM, Johnsson P, Hagemann-Jensen M, Hartmanis L, Faridani OR, Reinius B, Segerstolpe Å, Rivera CM, <B>Ren B</B>, Sandberg R. Genomic encoding of transcriptional burst kinetics. Nature. 2019 Jan 2. doi: 10.1038/s41586-018-0836-1.
</li></ul>
<ul><li>Patel L, Kang R, Rosenberg SC, Qiu Y, Raviram R, Chee S, Hu R, <B>Ren B</B>, Cole F, Corbett KD. Dynamic reorganization of the genome shapes the recombination landscape in meiotic prophase. Nat Struct Mol Biol. 2019 Mar;26(3):164-174. PMID: 30778236
</li></ul>
<ul><li>Juric I, Yu M, Abnousi A, Raviram R, Fang R, Zhao Y, Zhang Y, Qiu Y, Yang Y, Li Y, <B>Ren B</B>*, Hu M*. MAPS: Model-based analysis of long-range chromatin interactions from PLAC-seq and HiChIP experiments. PLoS Comput Biol. 2019 Apr 15;15(4):e1006982. (* co-corresponding author)
</li></ul>
<ul><li>Chaisson MJP, Sanders AD, Zhao X, Malhotra A, Porubsky D, Rausch T, Gardner EJ, Rodriguez OL, Guo L, Collins RL, Fan X, Wen J, Handsaker RE, Fairley S, Kronenberg ZN, Kong X, Hormozdiari F, Lee D, Wenger AM, Hastie AR, Antaki D, Anantharaman T, Audano PA, Brand H, Cantsilieris S, Cao H, Cerveira E, Chen C, Chen X, Chin CS, Chong Z, Chuang NT, Lambert CC, Church DM, Clarke L, Farrell A, Flores J, Galeev T, Gorkin DU, Gujral M, Guryev V, Heaton WH, Korlach J, Kumar S, Kwon JY, Lam ET, Lee JE, Lee J, Lee WP, Lee SP, Li S, Marks P, Viaud-Martinez K, Meiers S, Munson KM, Navarro FCP, Nelson BJ, Nodzak C, Noor A, Kyriazopoulou-Panagiotopoulou S, Pang AWC, Qiu Y, Rosanio G, Ryan M, Stütz A, Spierings DCJ, Ward A, Welch AE, Xiao M, Xu W, Zhang C, Zhu Q, Zheng-Bradley X, Lowy E, Yakneen S, McCarroll S, Jun G, Ding L, Koh CL, <B>Ren B</B>, Flicek P, Chen K, Gerstein MB, Kwok PY, Lansdorp PM, Marth GT, Sebat J, Shi X, Bashir A, Ye K, Devine SE, Talkowski ME, Mills RE, Marschall T, Korbel JO, Eichler EE, Lee C. Multi-platform discovery of haplotype-resolved structural variation in human genomes. Nat Commun. 2019 Apr 16;10(1):1784.
</li></ul>
<ul><li>Chowdhry S, Zanca C, Rajkumar U, Koga T, Diao Y, Raviram R, Liu F, Turner K, Yang H, Brunk E, Bi J, Furnari F, Bafna V, <B>Ren B</B>, Mischel PS. NAD metabolic dependency in cancer is shaped by gene amplification and enhancer remodelling. Nature. 2019 Apr 24. [Epub ahead of print]
</li></ul>
<ul><li>Greenwald WW, Chiou J, Yan J, Qiu Y, Dai N, Wang A, Nariai N, Aylward A, Han JY, Kadakia N, Regue L, Okino ML, Drees F, Kramer D, Vinckier N, Minichiello L, Gorkin D, Avruch J, Frazer KA, Sander M, <B>Ren B</B>, Gaulton KJ. Pancreatic islet chromatin accessibility and conformation reveals distal enhancer networks of type 2 diabetes risk. Nat Commun. 2019 May 7;10(1):2078. 
</li></ul>
<ul><li>Nechemia-Arbely Y, Miga KH, Shoshani O, Aslanian A, McMahon MA, Lee AY, Fachinetti D, Yates JR 3rd, <B>Ren B</B>, Cleveland DW. DNA replication acts as an error correction mechanism to maintain centromere identity by restricting CENP-A to centromeres. Nat Cell Biol. 21(6):743-754. 
</li></ul>
<ul><li>Yang Y, Zhang Y, <B>Ren B</B>, Dixon JR, Ma J. Comparing 3D Genome Organization in Multiple Species Using Phylo-HMRF. Cell Syst. 2019 Jun 26;8(6):494-505.e14. 
</li></ul>
<ul><li>Li G, Liu Y, Zhang Y, Kubo N, Yu M, Fang R, Kellis M, <B>Ren B</B>. Joint profiling of DNA methylation and chromatin architecture in single cells. Nat Methods. 2019 Aug 5. [Epub ahead of print]
</li></ul>
<ul><li>Dall'Agnese A, Caputo L, Nicoletti C, di Iulio J, Schmitt A, Gatto S, Diao Y, Ye Z, Forcato M, Perera R, Bicciato S, Telenti A, <B>Ren B</B>, Puri PL. Transcription Factor-Directed Re-wiring of Chromatin Architecture for Somatic Cell Nuclear Reprogramming toward trans-Differentiation. Mol Cell. 2019 Aug 20. [Epub ahead of print]. PMID: 31519520
</li></ul>
<ul><li>Zhang Y, Li T, Preissl S, Amaral ML, Grinstein JD, Farah EN, Destici E, Qiu Y, Hu R, Lee AY, Chee S, Ma K, Ye Z, Zhu Q, Huang H, Fang R, Yu L, Izpisua Belmonte JC, Wu J, Evans SM, Chi NC, <B>Ren B</B>. Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells. Nat Genet. Sep;51(9):1380-1388. Epub 2019 Aug 19.
</li></ul>
<ul><li>Jung I, Schmitt A, Diao Y, Lee AJ, Liu T, Yang D, Tan C, Eom J, Chan M, Chee S, Chiang Z, Kim C, Masliah E, Barr CL, Li B, Kuan S, Kim D and <B>Ren B</B>.  A compendium of promoter-centered long-range chromatin interactions in the human genome.  Nat Genet, Sept 9, 2019 [Epub ahead of print]
</li></ul>
<ul><li>Wu S, Turner KM, Nguyen N, Raviram R, Erb M, Santini J, Luebeck J, Rajkumar U, Diao Y, Li B, Zhang W, Jameson N, Corces MR, Granja JM, Chen X, Coruh C, Abnousi A, Houston J, Ye Z, Hu R, Yu M, Kim H, Law JA, Verhaak RGW, Hu M, Furnari FB, Chang HY, <B>Ren B</B>, Bafna V, Mischel PS. Circular extrachromosomal DNA promotes oncogene expression and chromatin accessibility. Nature 2019 Nov 20. doi: 10.1038/s41586-019-1763-5. 
</li></ul>
<ul><li>Zhu C, Yu M, Huang H, Juric I, Abnousi A, Hu R, Lucero J, Behrens MM, Hu M, and <B>Ren B</B>. An ultra high-throughput method for combined analysis of accessible chromatin and transcriptome in single cells. Nature Structural & Molecular Biology, Nov 6. doi: 10.1038 (publication ahead of print)
</li></ul>
<ul><li>Zhang D, Tang Z, Huang H, Zhou G, Cui C, Weng Y, Liu W, Kim S, Lee S, Perez-Neut M, Ding J, Czyz D, Hu R, Ye Z, He M, Zheng YG, Shuman HA, Dai L, <B>Ren B</B>, Roeder RG, Becker L, Zhao Y. Metabolic regulation of gene expression by histone lactylation. Nature. 574(7779):575-580
</li></ul>
<ul><li>Chung CY, Ma Z, Dravis C, Preissl S, Poirion O, Luna G, Hou X, Giraddi RR, <B>Ren B</B>, Wahl GM. Single-Cell Chromatin Analysis of Mammary Gland Development Reveals Cell-State Transcriptional Regulators and Lineage Relationships. Cell Rep. 2019 Oct 8;29(2):495-510
</li></ul>
<ul><li>Nott A, Holtman IR, Coufal NG, Schlachetzki JCM, Yu M, Hu R, Han CZ, Pena M, Xiao J, Wu Y, Keulen Z, Pasillas MP, O'Connor C, Nickl CK, Schafer ST, Shen Z, Rissman RA, Brewer JB, Gosselin D, Gonda DD, Levy ML, Rosenfeld MG, McVicker G, Gage FH, <B>Ren B</B>, Glass CK. Brain cell type-specific enhancer-promoter interactome maps and disease risk association.  Science 2019 Nov 14. pii: eaay0793.
</li></ul>
<ul><li>Wells A, Heckerman D, Torkamani A, Yin L, Sebat J, <B>Ren B</B>, Telenti A, di Iulio J. Ranking of non-coding pathogenic variants and putative essential regions of the human genome. Nat Commun. 2019 Nov 20;10(1):5241. doi: 10.1038/s41467-019-13212-3.
</li></ul>
<ul><li>Gorkin DU, Qiu Y, Hu M, Fletez-Brant K, Liu T, Schmitt AD, Noor A, Chiou J, Gaulton KJ, Sebat J, Li Y, Hansen KD, <B>Ren B</B>. Common DNA sequence variation influences 3-dimensional conformation of the human genome. Genome Biol. 2019 Nov 28;20(1):255. doi: 10.1186/s13059-019-1855-4.
</li></ul>
<p><h3>2018</h3></p>
<ul><li>Yan J, Chen SA, Local A, Liu T, Qiu Y, Dorighi KM, Preissl S, Rivera CM, Wang C, Ye Z, Ge K, Hu M, Wysocka J, <B>Ren B</B>. Histone H3 lysine 4 monomethylation modulates long- range chromatin interactions at enhancers. Cell Research. 28(3):387. Epub 2018 Jan 9.
</li></ul>
<ul><li>O'Brien LL, Guo Q, Bahrami-Samani E, Park JS, Hasso SM, Lee YJ, Fang A, Kim AD, Guo J, Hong TM, Peterson KA, Lozanoff S, Raviram R, <B>Ren B</B>, Fogelgren B, Smith AD, Valouev A, McMahon AP. Transcriptional regulatory control of mammalian nephron progenitors revealed by multi-factor cistromic analysis and genetic studies. PLoS Genet. 14(1):e1007181. 
</li></ul>
<ul><li>Preissl S, Fang R, Huang H, Zhao Y, Raviram R, Gorkin DU, Zhang Y, Sos BC, Afzal V, Dickel DE, Kuan S, Visel A, Pennacchio LA, Zhang K, <B>Ren B</B>. Single-nucleus analysis of accessible chromatin in developing mouse forebrain reveals cell-type-specific transcriptional regulation. Nat Neurosci. 21(3):432-439. Epub 2018 Feb 12.
</li></ul>
<ul><li>di Iulio J, Bartha I, Wong EHM, Yu HC, Lavrenko V, Yang D, Jung I, Hicks MA, Shah N, Kirkness EF, Fabani MM, Biggs WH, <B>Ren B</B>, Venter JC, Telenti A. The human noncoding genome defined by genetic diversity. Nat Genet. 50(3):333-337. Epub 2018 Feb 26.
</li></ul>
<ul><li>Koga T, Li B, Figueroa JM, <B>Ren B</B>, Chen CC, Carter BS, Furnari FB. Mapping of genomic EGFRvIII deletions in glioblastoma: insight into rearrangement mechanisms and biomarker development. Neuro Oncol. 2018 Apr 12. doi: 10.1093/neuonc/noy058. [Epub ahead of print]
</li></ul>
<ul><li>Fei J, Ishii H, Hoeksema MA, Meitinger F, Kassavetis GA, Glass CK, <B>Ren B</B>, and Kadonaga JT, NDF, a nucleosome destabilizing factor that facilitates transcription through 65 nucleosomes.  Genes & Dev. 2018 May 14. doi: 10.1101/gad.313973.118
</li></ul>
<ul><li>Link VM, Duttke SH, Chun HB, Holtman IR, Westin E, Hoeksema MA, Abe Y, Skola D,  Romanoski CE, Tao J, Fonseca G, Troutman TD, Spann N, Strid T, Sakai M, Yu M, Hu R, Fang RX, Metzler D, <B>Ren B</B>, and Glass CK. Analysis of genetically diverse macrophages reveals local and domain-wide mechanisms that control transcription factor binding and function. Cell 2018 May 5. pii: S0092-8674(18)30511-7
</li></ul>
<ul><li>Plouffe SW, Lin KC, Moore JL 3rd, Tan FE, Ma S, Ye Z, Qiu Y, <B>Ren B</B>, Guan KL. The Hippo pathway effector proteins YAP and TAZ have both distinct and overlapping functions in the cell. J Biol Chem. 2018 May 25. [Epub ahead of print] PMID: 29802201
</li></ul>
<ul><li>Meng Z, Qiu Y, Lin KC, Kumar A, Placone JK, Fang C, Wang KC, Lu S, Pan M, Hong AW, Moroishi T, Luo M, Plouffe SW, Diao Y, Ye Z, Park HW, Wang X, Yu FX, Chien S, Wang CY, <B>Ren B</B>, Engler AJ & Guan KL, RAP2 mediates mechanoresponses of the Hippo pathway.  Nature 560(7720):655-660
</li></ul>
<ul><li>Peng Q, Lu S, Shi Y, Pan Y, Limsakul P, Chernov AV, Qiu J, Chai X, Shi Y, Wang P, Ji Y, Li YJ, Strongin AY, Verkhusha VV, Izpisua Belmonte JC, <B>Ren B</B>, Wang Y, Chien S, Wang Y. Coordinated histone modifications and chromatin reorganization in a single cell revealed by FRET biosensors. Proc Natl Acad Sci U S A. 2018 Dec 11;115(50):E11681-E11690. Epub 2018 Nov 26.
</li></ul>
<ul><li>Jiang Q, Isquith J, Zipeto MA, Diep RH, Pham J, Delos Santos N, Reynoso E, Chau J, Leu H, Lazzari E, Melese E, Ma W, Fang R, Minden M, Morris S, <B>Ren B</B>, Pineda G, Holm F, Jamieson C. Hyper-Editing of Cell-Cycle Regulatory and Tumor Suppressor RNA Promotes Malignant Progenitor Propagation.  Cancer Cell. 2018 Dec 11. pii: S1535-6108(18)30538-5. doi: 10.1016/j.ccell.2018.11.017. [Epub ahead of print]
</li></ul>
<p><h3>2017</h3></p>
<ul><li>Turner KM, Deshpande V, Beyter D, Koga T, Rusert J, Lee C, Li B, Arden K, <B>Ren B</B>, Nathanson DA, Kornblum HI, Taylor MD, Kaushal S, Cavenee WK, Wechsler-Reya R, Furnari FB, Vandenberg SR, Rao PN, Wahl GM, Bafna V, Mischel PS. Extrachromosomal oncogene amplification drives tumour evolution and genetic heterogeneity. Nature. 543(7643):122-125. Epub 2017 Feb 8. PMID: 28178237
</li></ul>
<ul><li>He Y, Gorkin DU, Dickel DE, Nery JR, Castanon RG, Lee AY, Shen Y, Visel A, Pennacchio LA, <B>Ren B</B>, Ecker JR. Improved regulatory element prediction based on tissue-specific local epigenomic signatures. Proc Natl Acad Sci U S A. 114(9):E1633-E1640. Epub 2017 Feb 13. PMID: 28193886
</li></ul>
<ul><li>Xiong X, Zhang Y, Yan J, Jain S, Chee S, <B>Ren B</B>*, Zhao H*. A Scalable Epitope Tagging Approach for High Throughput ChIP-Seq Analysis. ACS Synth Biol. acssynbio.6b00358. PMID: 28215080 (*co-corresponding author)
</li></ul>
<ul><li>Nechemia-Arbely Y, Fachinetti D, Miga KH, Sekulic N, Soni GV, Kim DH, Wong AK, Lee AY, Nguyen K, Dekker C, <B>Ren B</B>, Black BE, Cleveland DW. Human centromeric CENP-A chromatin is a homotypic, octameric nucleosome at all cell cycle points. J Cell Biol. 216(3):607-621. Epub 2017 Feb 24. PMID: 28235947
</li></ul>
<ul><li>DeBoever C, Li H, Jakubosky D, Benaglio P, Reyna J, Olson KM, Huang H, Biggs W, Sandoval E, D'Antonio M, Jepsen K, Matsui H, Arias A, <B>Ren B</B>, Nariai N, Smith EN, D'Antonio-Chronowska A, Farley EK, Frazer KA. Large-Scale Profiling Reveals the Influence of Genetic Variation on Gene Expression in Human Induced Pluripotent Stem Cells. Cell Stem Cell. 20(4):533-546.e7. PMID: 28388430
</li></ul>
<ul><li>Diao Y, Fang R, Li B, Meng Z, Yu J, Qiu Y, Lin KC, Huang H, Liu T, Marina RJ, Jung I, Shen Y, Guan KL, <B>Ren B</B>. A tiling-deletion-based genetic screen for cis-regulatory element identification in mammalian cells. Nat Methods. nmeth.4264. PMID: 28417999
</li></ul>
<ul><li>Martin JS, Xu Z, Reiner AP, Mohlke KL, Sullivan P, <B>Ren B</B>, Hu M, Li Y. HUGIn: Hi-C Unifying Genomic Interrogator. Bioinformatics. 2017 Jun 5. [Epub ahead of print]
</li></ul>
<ul><li>Rosa-Garrido M, Chapski DJ, Schmitt AD, Kimball TH, Karbassi E, Monte E, Balderas E, Pellegrini M, Shih TT, Soehalim E, Liem DA, Ping P, Galjart NJ, Ren S, Wang Y, <B>Ren B</B>, Vondriska TM. High Resolution Mapping of Chromatin Conformation in Cardiac Myocytes Reveals Structural Remodeling of the Epigenome in Heart Failure. Circulation. 2017 Aug 11. pii: CIRCULATIONAHA.117.029430. PMID: 28802249
</li></ul>
<ul><li>Lee J, Liao R, Wang G, Yang BH, Luo X, Varki NM, Qiu SJ, <B>Ren B</B>, Fu W, Feng GS. Preventive Inhibition of Liver Tumorigenesis by Systemic Activation of Innate Immune Functions. Cell Rep. 2017 Nov 14;21(7):1870-1882. 
</li></ul>
<ul><li>Local A, Huang H, Albuquerque CP, Singh N, Lee AY, Wang W, Wang C, Hsia JE, Shiau AK, Ge K, Corbett KD, Wang D, Zhou H, <B>Ren B</B>. Identification of H3K4me1-associated proteins at mammalian enhancers. Nat Genet. 50(1):73-82. Epub 2017 Dec 18.
</li></ul>
<p><h3>2016</h3></p>
<ul><li>Diao Y, Li B, Meng Z, Jung I, Lee A, Dixon J, Maliskova L, Guan KL, Shen Y* and <B>B Ren</B>*, A new class of temporarily phenotypic enhancers identified by CRISPR/Cas9 mediated genetic screening. Genome Research, 26 (3), 397-405  (* co-corresponding author)
</li></ul>
<ul><li>Dahl JA*, Jung I, Aanes H, Greggains GD, Manaf A, Lerdrup M, Li G, Kuan S, Li B, Lee AY, Preissl S, Jermstad I, Haugen MH, Suganthan R, Bjørås M, Hansen K, Dalen KT, Fedorcsak P, <B>Ren B</B>*, Klungland A*. Broad histone H3K4me3 domains in mouse oocytes modulate maternal to zygotic transition. Nature.  537(7621):548-552 (* co-corresponding authors).  
</li></ul>
<ul><li>Dickel DE, Barozzi I, Zhu Y, Fukuda-Yuzawa Y, Osterwalder M, Mannion BJ, May D, Spurrell CH, Plajzer-Frick I, Pickle CS, Lee E, Garvin TH, Kato M, Akiyama JA, Afzal V, Lee AY, Gorkin DU, <B>Ren B</B>, Rubin EM, Visel A, Pennacchio LA. Genome-wide compendium and functional assessment of in vivo heart enhancers. Nat Commun. 7:12923.
</li></ul>
<ul><li>Schmitt AD, Hu M, Jung I, Xu Z, Qiu Y, Tan CL, Li Y, Lin S, Lin Y, Barr CL, <B>Ren B</B>. A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome. Cell Rep.17(8):2042-2059.
</li></ul>
<ul><li>Fang R, Yu M, Li G, Chee S, Liu T, Schmitt AD, <B>Ren B</B>. Mapping of long-range chromatin interactions by proximity ligation-assisted ChIP-seq. Cell Res. 26(12):1345-1348. 
</li></ul>
<p><h3>2015</h3></p>
<ul><li>Chen Y, Chen C, Zhang Z, Liu CC, Johnson ME, Espinoza CA, Edsall LE, <B>Ren B</B>, Zhou XJ, Grant SF, Wells AD, Chen L (2015) DNA binding by FOXP3 domain-swapped dimer suggests mechanisms of long-range chromosomal interactions. Nucleic Acids Res. 2015 Jan 7. pii: gku1373. [Epub ahead of print]
</li></ul>
<ul><li>JR. Dixon, I Jung, S Selvaraj, Y Shen, JE Antosiewicz-Bourget, AY Lee, Z Ye, A Kim, N Rajagopal, W Xie, J Liang, H Zhao, VV Lobanenkov, JE Ecker, JA Thomson, and <B>B Ren</B> (2015) Chromatin Architecture Reorganization during Stem Cell Differentiation. Nature, 518(7539):331-6.
</li></ul>
<ul><li>D Leung, I Jung, N Rajagopal, A Schmitt, S Selvaraj, AY Lee, CA Yen, S Lin, Y Lin, Y Qiu, W Xie, F Yue, M Hariharan, P Ray, S Kuan, L Edsall, H Yang, NC Chi, MQ Zhang, JR Ecker and <B>B Ren</B> (2015) Integrative analysis of haplotype-resolved epigenomes across human tissues.  Nature, 518(7539):350-4
</li></ul>
<ul><li>Roadmap Epigenomics Consortium, Kundaje A, Meuleman W, Ernst J, Bilenky M, Yen A, Heravi-Moussavi A, Kheradpour P, Zhang Z, Wang J, Ziller MJ, Amin V, Whitaker JW, Schultz MD, Ward LD, Sarkar A, Quon G, Sandstrom RS, Eaton ML, Wu YC, Pfenning AR, Wang X, Claussnitzer M, Liu Y, Coarfa C, Harris RA, Shoresh N, Epstein CB, Gjoneska E, Leung D, Xie W, Hawkins RD, Lister R, Hong C, Gascard P, Mungall AJ, Moore R, Chuah E, Tam A, Canfield TK, Hansen RS, Kaul R, Sabo PJ, Bansal MS, Carles A, Dixon JR, Farh KH, Feizi S, Karlic R, Kim AR, Kulkarni A, Li D, Lowdon R, Elliott G, Mercer TR, Neph SJ, Onuchic V, Polak P, Rajagopal N, Ray P, Sallari RC, Siebenthall KT, Sinnott-Armstrong NA, Stevens M, Thurman RE, Wu J, Zhang B, Zhou X, Beaudet AE, Boyer LA, De Jager PL, Farnham PJ, Fisher SJ, Haussler D, Jones SJ, Li W, Marra MA, McManus MT, Sunyaev S, Thomson JA, Tlsty TD, Tsai LH, Wang W, Waterland RA, Zhang MQ, Chadwick LH, Bernstein BE, Costello JF, Ecker JR, Hirst M, Meissner A, Milosavljevic A, <B>Ren B</B>, Stamatoyannopoulos JA, Wang T, Kellis M. Integrative analysis of 111 reference human epigenomes. Nature. 518(7539):317-30
</li></ul>
<ul><li>A Wang, F Yue, Y Li, R Xie, T Harper, NA Patel, K Muth, J Palmer, Y Qiu, J Wang, DK Lam, JC Raum, DA Stoffers, <B>B Ren</B>*, M Sander* (2015) Developmental Competence Encoded in Enhancer Chromatin, Cell Stem Cell, 16(4):386-99 (* co-corresponding author)
</li></ul>
<ul><li>Wu J, Okamura D, Li M, Suzuki K, Luo C, Ma L, He Y, Li Z, Benner C, Tamura I, Krause MN, Nery JR, Du T, Zhang Z, Hishida T, Takahashi Y, Aizawa E, Kim NY, Lajara J, Guillen P, Campistol JM, Esteban CR, Ross PJ, Saghatelian A, <B>Ren B</B>, Ecker JR, Izpisua Belmonte JC. (2015) An alternative pluripotent state confers interspecies chimaeric competency. Nature. 521(7552):316-21. 
</li></ul>
<ul><li>Ishii H, Kadonaga JT*, <B>Ren B</B>*. MPE-seq, a new method for the genome-wide analysis of chromatin structure. Proc Natl Acad Sci U S A. 2015 Jul 7;112(27):E3457-65. doi: 10.1073/pnas.1424804112. Epub 2015 Jun 15. (co-corresponding author)
</li></ul>
<ul><li>Schultz MD, He Y, Whitaker JW, Hariharan M, Mukamel EA, Leung D, Rajagopal N, Nery JR, Urich MA, Chen H, Lin S, Lin Y, Jung I, Schmitt AD, Selvaraj S, <B>Ren B</B>, Sejnowski TJ, Wang W, Ecker JR. Human body epigenome maps reveal noncanonical DNA methylation variation. Nature. 2015 Jul 9;523(7559):212-6. doi: 10.1038/nature14465. Epub 2015 Jun 1.
</li></ul>
<ul><li>Guo Y, Xu Q, Canzio D, Shou J, Li J, Gorkin DU, Jung I, Wu H, Zhai Y, Tang Y, Lu Y, Wu Y, Jia Z, Li W, Zhang MQ, <B>Ren B</B>, Krainer AR, Maniatis T, Wu Q. CRISPR Inversion of CTCF Sites Alters Genome Topology and Enhancer/Promoter Function. Cell. 2015 Aug 13;162(4):900-10. doi: 10.1016/j.cell.2015.07.038.
</li></ul>
<ul><li>Pugacheva EM, Rivero-Hinojosa S, Espinoza CA, Méndez-Catalá CF, Kang S, Suzuki T, Kosaka-Suzuki N, Robinson S, Nagarajan V, Ye Z, Boukaba A, Rasko JE, Strunnikov AV, Loukinov D, <B>Ren B</B>*, Lobanenkov VV*. Comparative analyses of CTCF and BORIS occupancies uncover two distinct classes of CTCF binding genomic regions. Genome Biol. 2015 Aug 14;16(1):161. [Epub ahead of print] (*co-corresponding author)
</li></ul>
<ul><li>Selvaraj S, Schmitt AD, Dixon JR and <B>Ren B</B>.  Complete haplotype phasing of the MHC and KIR loci with targeted HaploSeq.  BMC Genomics. Nov 5;16(1):900.
</li></ul>
<ul><li>Liu F, Hon GC, Villa GR, Turner KM, Ikegami S, Yang H, Ye Z, Li B, Kuan S, Lee AY, Zanca C, Wei B, Lucey G, Jenkins D, Zhang W, Barr CL, Furnari FB, Cloughesy TF, Yong WH, Gahman TC, Shiau AK, Cavenee WK, <B>Ren B</B>*, Mischel PS*. EGFR Mutation Promotes Glioblastoma through Epigenome and Transcription Factor Network Remodeling.  Molecular Cell, 60 (2), 307-318. (*co-corresponding author)
</li></ul>
<ul><li>Kanduri K, Tripathi S, Larjo A, Mannerström H, Ullah U, Lund R, Hawkins RD, <B>Ren B</B>, Lähdesmäki H, Lahesmaa R. Identification of global regulators of T-helper cell lineage specification. Genome Med. 2015 Nov 20;7(1):122. doi: 10.1186/s13073-015-0237-0.
</li></ul>
<p><h3>2014</h3></p>
<ul><li>Dai L, Peng C, Montellier E, Lu Z, Chen Y, Ishii H, Debernardi A, Buchou T, Rousseaux S, Jin F, Sabari BR, Deng Z, Allis CD, <B>Ren B</B>, Khochbin S, Zhao Y.(2014) Lysine 2-hydroxyisobutyrylation is a widely distributed active histone mark. Nat Chem Biol. 2014 Mar 30.
</li></ul>
<ul><li>Leung D, Du T, Wagner U, Xie W, Lee AY, Goyal P, Li Y, Szulwach KE, Jin P, Lorincz MC, <B>Ren B</B>. (2014) Regulation of DNA methylation turnover at LTR retrotransposons and imprinted loci by the histone methyltransferase Setdb1. Proc Natl Acad Sci U S A. 2014 May 6;111(18):6690-5. doi: 10.1073/pnas.1322273111. Epub 2014 Apr 22.
</li></ul>
<ul><li>Attanasio C, Nord AS, Zhu Y, Blow MJ, Biddie SC, Mendenhall EM, Dixon J, Wright C, Hosseini R, Akiyama JA, Holt A, Plajzer-Frick I, Shoukry M, Afzal V, <B>Ren B</B>, Bernstein BE, Rubin EM, Visel A, Pennacchio LA. (2014) Tissue-specific SMARCA4 binding at active and repressed regulatory elements during embryogenesis. Genome Res. 2014 Jun;24(6):920-9. doi: 10.1101/gr.168930.113. Epub 2014 Apr 21.
</li></ul>
<ul><li>Rajagopal N, Ernst J, Ray P, Wu J, Zhang M, Kellis M, <B>Ren B</B> (2014) Distinct and predictive histone lysine acetylation patterns at promoters, enhancers and gene bodies.  Genes, Genomes and Genetics, 4(11):2051-63
</li></ul>
<ul><li>Hon GC, Song C, Du T, Jin F, Selvaraj S, Lee AY, Yen C, Ye Z, Mao S, Wang B, Kuan S, Edsall LE, Zhao BS, Xu GL, He C*, <B>Ren B</B>* (2014) 5mC oxidation by Tet2 modulates enhancer activity and timing of transcriptome reprogramming during differentiation.  Molecular Cell, 56(2):286-97 (*co-corresponding authors)
</li></ul>
<ul><li>Yue F, ….. <B>Ren B</B>* and Mouse ENCODE Consortium (2014) Comparative encyclopedia of DNA elements in the mouse genome.  Nature 515(7527):355-64  (* one of seven co-corresponding authors, out of 146 total authors; lead the writing and analysis)
</li></ul>
<ul><li>Pope BD, Ryba T, Dileep V, Yue F, Wu W, Denas O, Vera DL, Wang Y, Hansen RS, Canfield TK, Thurman RE, Cheng Y, Gülsoy G, Dennis JH, Snyder MP, Stamatoyannopoulos JA, Taylor J, Hardison RC, Kahveci T, <B>Ren B</B> & Gilbert DM. (2014) Topologically-associating domains are stable units of replication-timing regulation.  Nature, 515(7527):402-5.
</li></ul>
<ul><li>Li Y, Rivera CM, Ishii H, Jin F, Selvaraj S, Lee AY, Dixon JR, <B>Ren B</B> (2014) CRISPR Reveals a Distal Super-Enhancer Required for Sox2 Expression in Mouse Embryonic Stem Cells. PLoS One. 9(12):e114485. 
</li></ul>
<p><h3>2013</h3></p>
<ul><li>Sawan C, Hernandez-Vargas H, Murr R, Lopez F, Vaissière T, Ghantous AY, Cuenin C, Imbert J, Wang ZQ, <B>Ren B</B>, Herceg Z. (2013) HAT cofactor Trrap Maintains Self-renewal and Restricts Differentiation of Embryonic Stem Cells. Stem Cells. 31(5):979-91. 2013 Jan 29. [Epub ahead of print] 
</li></ul>
<ul><li>Hu M, Deng K, Qin Z, Dixon J, Selvaraj S, Fang J, <B>Ren B</B> and Liu JS (2013) Bayesian Inference of Spatial Organizations of Chromosomes, PLoS Comput Biol. 2013 Jan;9(1):e1002893.
</li></ul>
<ul><li>Zhang L, Szulwach KE, Hon GC, Song CX, Park B, Yu M, Lu X, Dai Q, Wang X, Street CR, Tan H, Min JH, <B>Ren B</B>, Jin P, He C. (2013) Tet-mediated covalent labelling of 5-methylcytosine for its genome-wide detection and sequencing. Nat Commun. 4:1517.
</li></ul>
<ul><li>Rajagopal N, Xie W, Li Y, Wagner U, Wang W, Stamatoyannopoulos J, Ernst J, Kellis M, and <B>Ren B</B>. (2013) RFECS: A Random-Forest Based Algorithm for Enhancer Identification From Chromatin State. PLoS Computational Biology, 9(3):e1002968.
</li></ul>
<ul><li>Xie W, Schultz M, Lister R, Hou Z, Rajagopal N, Ray P, Whitaker JW, Tian T, Hawkins RD, Leung D, Yang H, Wang T, Lee AY, Swanson SA, Zhang J, Zhu Y, Kim A, Nery J, Urich MA, Kuan S, Yen C, Klugman S, Yu P, Suknuntha K, Propson NE, Chen H, Edsall LE, Wagner U, Li Y, Ye Z, Kulkarni A, Xuan Z, Chung W, Chi NC, Antosiewicz-Bourget J, Slukvin I, Stewart R, Zhang MQ*, Wang W*, Thomson JA*, Ecker JR*, and <B>Ren B</B>* (2013) Epigenomic Analysis of Multi-lineage Differentiation of Human Embryonic Stem Cells, Cell 153(5):1134-48 (*co-corresponding authors)
</li></ul>
<ul><li>Won KJ, Zhang X, Wang T, Ding B, Raha D, Snyder M, <B>Ren B</B>, Wang W. (2013) Comparative annotation of functional regions in the human genome using epigenomic data. Nucleic Acids Res. 41(8):4423-4432. Epub 2013 Mar 12.
</li></ul>
<ul><li>Chang KN, Zhong S, Weirauch MT, Hon G, Pelizzola M, Li H, Huang SS, Schmitz RJ, Urich MA, Kuo D, Nery JR, Qiao H, Yang A, Jamali A, Chen H, Ideker T, <B>Ren B</B>, Bar-Joseph Z, Hughes TR, Ecker JR. (2013) Temporal transcriptional response to ethylene gas drives growth hormone cross-regulation in Arabidopsis. Elife. 2:e00675.
</li></ul>
<ul><li>Hawkins RD, Larjo A, Tripathi SK, Wagner U, Luu Y, Lönnberg T, Raghav SK, Lee LK, Lund R, <B>Ren B</B>, Lähdesmäki H, Lahesmaa R. (2013) Global Chromatin State Analysis Reveals Lineage-Specific Enhancers during the Initiation of Human T helper 1 and T helper 2 Cell Polarization. Immunity. 38(6):1271-84. 
</li></ul>
<ul><li>Hon GC, Rajagopal N, Shen Y, McCleary DF, Yue F, Dang MD, and <B>Ren B</B> (2013), Epigenetic memory at embryonic enhancers identified from adult tissue methylomes.  Nature Genetics, 45(10):1198-206. Publication online September 1st, 2013, 
</li></ul>
<ul><li>Jin F, Li Y, Dixon JR, Selvaraj S, Ye Z, Lee AY, Yen AC, Schmitt AC, Espinoza C, and <B>Ren B</B> (2013) A High-resolution Map of Signaling-dependent, Dynamic Chromatin Interactome in Human Cells.  Nature, 503(7475):290-4
</li></ul>
<ul><li>Wang T, Liu J, Shen L, Tonti-Filippini J, Zhu Y, Jia H, Lister R, Whitaker JW, Ecker JR, Millar AH, <B>Ren B</B>, Wang W. (2013) STAR: an integrated solution to management and visualization of sequencing data. Bioinformatics. Oct 23. [Epub ahead of print]
</li></ul>
<ul><li>Attanasio C, Nord AS, Zhu Y, Blow MJ, Li Z, Liberton DK, Morrison H, Plajzer-Frick I, Holt A, Hosseini R, Phouanenavong S, Akiyama JA, Shoukry M, Afzal V, Rubin EM, FitzPatrick DR, <B>Ren B</B>, Hallgrímsson B, Pennacchio LA, Visel A. (2013) Fine tuning of craniofacial morphology by distant-acting enhancers. Science, 342:1241006
</li></ul>
<ul><li>Selvaraj S, Dixon JR, Bansal V and <B>Ren, B</B> (2013) Whole-genome haplotype reconstruction using proximity-ligation and shotgun sequencing.  Nature Biotechnology, 31(12):1111-8.
</li></ul>
<ul><li>Wang X, Lu Z, Gomez A, Hon GC, Yue Y, Han D, Fu Y, Parisien M, Dai Q, Jia G, <B>Ren B</B>, Pan T, He C. N6-methyladenosine-dependent regulation of messenger RNA stability. Nature. 2013 Nov 27. doi: 10.1038/nature12730.
</li></ul>
<ul><li>Looney T, Lee JH, Zhang L, Chen CH, Chari S, Mao FF, Pelizzola M, Zhang L, Lister R, Baker S, Fernandes C, Gaetz J, Foshay K, Clift K, Zhang Z, Li WQ, Vallender E, Wagner U, Qin JY, Michelini K, Bugarija B, Park D, Aryee E, Stricker T, White K, <B>Ren B</B>, Schroth G, Ecker J, Xiang AP, Zhou J, Lahn B. Systematic mapping of occluded genes by cell fusion reveals prevalence and stability of cis- mediated silencing in somatic cells. Genome Res. 2013 Dec 5. [Epub ahead of print]
</li></ul>
<ul><li>Zuin J, Dixon JR, van der Reijdena MI, Ye Z, Kolovosa P, Brouwer RW, van de Corputa MP, van de Werkena HJ, Knochh TA, van IJckenf WF, Grosvelda FG, <B>Ren B</B>*, and Wendt KS* (2013) Cohesin and CTCF differentially affect chromatin architecture and gene expression in human cells.  PNAS (published online Dec 13, 2013)  (* co-corresponding authors)
</li></ul>
<p><h3>2012</h3></p>
<ul><li>Jishage M, Malik S, Wagner U, Uberheide B, Ishihama Y, Hu X, Chait BT, Gnatt A, <B>Ren B</B>, Roeder RG. (2012) Transcriptional regulation by Pol II(G) involving mediator and competitive interactions of Gdown1 and TFIIF with Pol II. Mol Cell. 45(1):51-63.
</li></ul>
<ul><li>Xie W, Barr CL*, Kim A, Yue F, Lee AY, Eubanks J, Dempster EL, <B>Ren B</B>.* (2012) Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome. Cell. 148(4):816-31. (*co-corresponding authors) 
</li></ul>
<ul><li>Hunkapiller J, Shen Y, Diaz A, Cagney G, McCleary D, Ramalho-Santos M, Krogan N, <B>Ren B</B>, Song JS, Reiter JF. Polycomb-like 3 promotes polycomb repressive complex 2 binding to CpG islands and embryonic stem cell self-renewal. PLoS Genet. 2012 Mar;8(3):e1002576. Epub 2012 Mar 15. 
</li></ul>
<ul><li>Dixon JR, Selvaraj S, Yue F, Kim A, Li Y, Shen Y, Hu M, Liu JS, <B>Ren B</B>. (2012) Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature. 485(7398):376-80 
</li></ul>
<ul><li>Yu M, Hon GC, Szulwach KE, Song C, Zhang L, Kim A, Li X, Dai Q, Park B, Min JH, Jin P *, <B>Ren B</B>*, He C * (2012). Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome. Cell, 149(6):1368-80. (*co-corresponding authors)
</li></ul>
<ul><li>Won KJ, Xu Z, Zhang X, Whitaker JW, Shoemaker R, <B>Ren B</B>, Xu Y, Wang W. (2012) Global identification of transcriptional regulators of pluripotency and differentiation in embryonic stem cells. Nucleic Acids Res. 40(17):8199-8209. 
</li></ul>
<ul><li>Lo MC, Peterson LF, Yan M, Cong X, Jin F, Shia WJ, Matsuura S, Ahn EY, Komeno Y, Ly M, Ommen HB, Chen IM, Hokland P, Willman CL, <B>Ren B</B>, Zhang DE. (2012) Combined gene expression and DNA occupancy profiling identifies potential therapeutic targets of t(8;21) AML. Blood. 120(7):1473-84.
</li></ul>
<ul><li>Shen Y, Yue F, McCleary DF, Ye Z, Edsall L, Kuan S, Wagner U, Dixon J, Lee L, Lobanenkov VV, <B>Ren B</B>. (2012) A map of the cis-regulatory sequences in the mouse genome. Nature, 488(7409):116-20. 
</li></ul>
<ul><li>Smallwood A, Hon GC, Jin F, Henry RE, Espinosa JM, <B>Ren B</B>. (2012) CBX3 regulates efficient RNA processing genome-wide. Genome Res. 2012 Jul 3. [Epub ahead of print] 
</li></ul>
<ul><li>Hu M, Deng K, Selvaraj S, Qin Z, <B>Ren B</B>, Liu JS. HiCNorm: removing biases in Hi-C data via Poisson regression. Bioinformatics. 2012 Sep 27. [Epub ahead of print]
</li></ul>
<ul><li>Liu GH, Qu J, Suzuki K, Nivet E, Li M, Montserrat N, Yi F, Xu X, Ruiz S, Zhang W, Wagner U, Kim A, <B>Ren B</B>, Li Y, Goebl A, Kim J, Soligalla RD, Dubova I, Thompson J, Iii JY, Esteban CR, Sancho-Martinez I, Belmonte JC. (2012) Progressive degeneration of human neural stem cells caused by pathogenic LRRK2. Nature. 491: 603-607 
</li></ul>
<ul><li>Yu M, Hon GC, Szulwach KE, Song CX, Jin P, <B>Ren B</B>, He C. (2012) Tet-assisted bisulfite sequencing of 5-hydroxymethylcytosine. Nat Protoc. 2012 Nov 29;7(12):2159-70. doi: 10.1038/nprot.2012.137. Epub 2012 Nov 29.
</li></ul>
<p><h3>2011</h3></p>
<ul><li>Lister R, Pelizzola M, Kida YS, Hawkins RD, Nery JR, Hon G, Antosiewicz-Bourget J, O'Malley R, Castanon R, Klugman S, Downes M, Yu R, Stewart R, <B>Ren B</B>, Thomson JA, Evans RM, Ecker JR. (2011) Hotspots of aberrant epigenomic reprogramming in human induced pluripotent stem cells. Nature. 471(7336):68-73
</li></ul>
<ul><li>Harismendy O, Notani D, Rahim NG, Song S, Tanasa B, Heintzman ND, <B>Ren B</B>, Fu XD, Topol EJ, Rosenfeld MG, Frazer KA (2011) Multiple enhancers in the 9p21 gene desert contain regulatory variants associated with Coronary Artery Disease and Type 2 Diabetes risk.  Nature, 470(7333):264-8
</li></ul>
<ul><li>Jin, F, Li, Y, <B>Ren, B</B>* and Natarajan, R* (2011) PU.1 and C/EBPα synergistically program distinct response to NF-κB activation through establishing monocyte specific enhancers, Proc Natl Acad Sci U S A, 108(13):5290-5
</li></ul>
<ul><li>Nègre N, Brown CD, Ma L, Bristow CA, Miller SW, Wagner U, Kheradpour P, Eaton ML, Loriaux P, Sealfon R, Li Z, Ishii H, Spokony RF, Chen J, Hwang L, Cheng C, Auburn RP, Davis MB, Domanus M, Shah PK, Morrison CA, Zieba J, Suchy S, Senderowicz L, Victorsen A, Bild NA, Grundstad AJ, Hanley D, MacAlpine DM, Mannervik M, Venken K, Bellen H, White R, Gerstein M, Russell S, Grossman RL, <B>Ren B</B>, Posakony JW, Kellis M & White KP (2011) A cis-regulatory map of the Drosophila genome, Nature. 471(7339):527-31
</li></ul>
<ul><li>Macfarlan TS, Gifford WD, Agarwal S, Driscoll S, Lettieri K, Wang J, Andrews SE, Franco L, Rosenfeld MG, <B>Ren B</B>, Pfaff SL (2011). Endogenous retroviruses and neighboring genes are coordinately repressed by LSD1/KDM1A. Genes Dev. 594-607
</li></ul>
<ul><li>Galante PA, Parmigiani RB, Zhao Q, Caballero OL, de Souza JE, Navarro FC, Gerber AL, Nicolás MF, Salim AC, Silva AP, Edsall L, Devalle S, Almeida LG, Ye Z, Kuan S, Pinheiro DG, Tojal I, Pedigoni RG, de Sousa RG, Oliveira TY, de Paula MG, Ohno-Machado L, Kirkness EF, Levy S, da Silva WA Jr, Vasconcelos AT, <B>Ren B</B>, Zago MA, Strausberg RL, Simpson AJ, de Souza SJ, Camargo AA. Distinct patterns of somatic alterations in a lymphoblastoid and a tumor genome derived from the same individual. Nucleic Acids Res. 2011 Apr 14. [Epub ahead of print] 
</li></ul>
<ul><li>Boyapati A, <B>Ren B</B>, Zhang DE (2011). SERPINB13 is a novel RUNX1 target gene. Biochem Biophys Res Commun. 2011 Jun 23. [Epub ahead of print]
</li></ul>
<ul><li>Szulwach KE, Li X, Li Y, Song CX, Han JW, Kim S, Namburi S, Hermetz K, Kim JJ, Rudd MK, Yoon YS, <B>Ren B</B>, He C, Jin P. Integrating 5-hydroxymethylcytosine into the epigenomic landscape of human embryonic stem cells. PLoS Genet. 2011 Jun;7(6):e1002154. Epub 2011 Jun 23. 
</li></ul>
<ul><li>Tan, M, Luo, H, Lee, S, Jin, F, Yang JS, Montellier, E, Buchou, T, Cheng, Z, Rousseaux, S, Rajagopal, N, Lu, Z, Ye, Z, Zhu, Q, Wysocka, J, Ye, Y, Khochbin, S, <B>Ren, B</B>, Zhao Y (2011) Identification of 67 histone marks and histone lysine crotonylation as a new type of histone modification, Cell 146(6):1016-28
</li></ul>
<ul><li>Hawkins RD, Hon GC, Yang C, Antosiewicz-Bourget JE, Lee LK, Ngo QM, Klugman S, Ching KA, Edsall LE, Ye Z, Kuan S, Yu P, Liu H, Zhang X, Green RD, Lobanenkov VV, Stewart R, Thomson JA*, and <B>Ren B</B>.* (2011) Dynamic Chromatin States in Human ES Cells Reveal Potential Regulatory Sequences and Genes Involved in Pluripotency. Cell Research 21(10):1393-409.  (*co-corresponding author) 
</li></ul>
<ul><li>Hon GC, Hawkins RD, Caballero OL, Lo C, Lister R, Pelizzola M, Valsesia A, Ye Z,  Kuan S, Edsall LE, Camargo AA, Stevenson BJ, Ecker JR, Bafna V, Strausberg RL, Simpson AJ, and <B>Ren B</B> (2011) Global DNA Hypomethylation Coupled to Repressive Chromatin Domain Formation and Gene Silencing in Breast Cancer.  Genome Research, 2011 Dec 7. [Epub ahead of print]
</li></ul>
<p><h3>2010</h3></p>
<ul><li>Won, KJ, <B>Ren, B</B>, Wang, W. (2010) Genome-wide prediction of transcription factor binding sites using an integrated model. Genome Biology,11(1):R7
</li></ul>
<ul><li>Hawkins RD, Hon GC, Lee LK, Ngo Q, Lister R, Pelizzola M, Edsall LE, Kuan S, Luu Y, Klugman S, Antosiewicz-Bourget J, Ye Z, Espinoza C, Agarwahl S, Shen L, Ruotti V, Wang W, Stewart R, Thomson JA, Ecker JR, <B>Ren B</B>. (2010) Distinct epigenomic landscapes of pluripotent and lineage-committed human cells. Cell Stem Cell. 6(5):479-91
</li></ul>
<ul><li>Harris RA, Wang T, Coarfa C, Nagarajan RP, Hong C, Downey SL, Johnson BE, Fouse SD, Delaney A, Zhao Y, Olshen A, Ballinger T, Zhou X, Forsberg KJ, Gu J, Echipare L, O'Geen H, Lister R, Pelizzola M, Xi Y, Epstein CB, Bernstein BE, Hawkins RD, <B>Ren B</B>, Chung WY, Gu H, Bock C, Gnirke A, Zhang MQ, Haussler D, Ecker JR, Li W, Farnham PJ, Waterland RA, Meissner A, Marra MA, Hirst M, Milosavljevic A, Costello JF (2010).Comparison of sequencing-based methods to profile DNA methylation and identification of monoallelic epigenetic modifications. Nat Biotechnol. 28(10):1097-105. 
</li></ul>
<ul><li>Bazarov AV, van Sluis M, Hines WC, Bassett E, Beliveau A, Campeau E, Mukhopadhyay R, Lee WJ, Melodyev S, Zaslavsky Y, Lee L, Rodier F, Chicas A, Lowe SW, Benhattar J, <B>Ren B</B>, Campisi J, Yaswen P (2010) p16INK4a-mediated suppression of telomerase in normal and malignant human breast cells, Aging Cell, 9: 736-746
</li></ul>
<ul><li>Blow MJ, McCulley DJ, Li ZR, Zhang T, Akiyama JA, Holt A, Plajzer-Frick I, Shoukry M, Wright C, Chen F, Afzal V, Bristow J, <B>Ren B</B>, Black BL, Rubin EM, Visel A, Pennacchio LA (2010) ChIP-Seq identification of weakly conserved heart enhancers, Nature Genetics, 42: 806-810
</li></ul>
<ul><li>Egelhofer TA, Minoda A, Klugman S, Lee K, Kolasinska-Zwierz P, Alekseyenko AA, Cheung MS, Day DS, Gadel S, Gorchakov AA, Gu T, Kharchenko PV, Kuan S, Latorre I, Linder-Basso D, Luu Y, Ngo Q, Perry M, Rechtsteiner A, Riddle NC, Schwartz YB, Shanower GA, Vielle A, Ahringer J, Elgin SC, Kuroda MI, Pirrotta V, <B>Ren B</B>, Strome S, Park PJ, Karpen GH, Hawkins RD, Lieb JD. (2010)  An assessment of histone-modification antibody quality. Nat Struct Mol Biol. 2011, 18(1):91-3. 
</li></ul>
<ul><li>Zhao Q, Kirkness EF, Caballero OL, Galante PA, Parmigiani RB, Edshall L, Kuan S, Ye Z, Levy S, Vasconcelos AT, <B>Ren B</B>, de Souza SJ, Camargo AA, Simpson AJ, Strausberg RL. (2010) Systematic detection of putative tumor suppressor genes through the combined use of exome and transcriptome sequencing. Genome Biol. 11(11):R114. 
</li></ul>
<ul><li>The modENCODE Consortium, Roy S, Ernst J, Kharchenko PV, Kheradpour P, Negre N, Eaton ML, Landolin JM, Bristow CA, Ma L, Lin MF, Washietl S, Arshinoff BI, Ay F, Meyer PE, Robine N, Washington NL, Di Stefano L, Berezikov E, Brown CD, Candeias R, Carlson JW, Carr A, Jungreis I, Marbach D, Sealfon R, Tolstorukov MY, Will S, Alekseyenko AA, Artieri C, Booth BW, Brooks AN, Dai Q, Davis CA, Duff MO, Feng X, Gorchakov AA, Gu T, Henikoff JG, Kapranov P, Li R, Macalpine HK, Malone J, Minoda A, Nordman J, Okamura K, Perry M, Powell SK, Riddle NC, Sakai A, Samsonova A, Sandler JE, Schwartz YB, Sher N, Spokony R, Sturgill D, van Baren M, Wan KH, Yang L, Yu C, Feingold E, Good P, Guyer M, Lowdon R, Ahmad K, Andrews J, Berger B, Brenner SE, Brent MR, Cherbas L, Elgin SC, Gingeras TR, Grossman R, Hoskins RA, Kaufman TC, Kent W, Kuroda MI, Orr-Weaver T, Perrimon N, Pirrotta V, Posakony JW, <B>Ren B</B>, Russell S, Cherbas P, Graveley BR, Lewis S, Micklem G, Oliver B, Park PJ, Celniker SE, Henikoff S, Karpen GH, Lai EC, Macalpine DM, Stein LD, White KP, Kellis M. (2010) Identification of Functional Elements and Regulatory Circuits by Drosophila modENCODE. Science. 330(6012):1787-97
</li></ul>
<p><h3>2009</h3></p>
<ul><li>Heintzman, ND, Hon, GC, Hawkins, RD, Kheradpour, P, Stark, A, Stuart, RK, Harp, LF, Ye, Z, Ching, KC, Ching, CW, Antosiewicz, JE, Liu, H, Zhang, X, Green, RD, Stewart, R, Thomson, JA, Crawford, GE, Kellis, M and <B>Ren, B</B>. (2009) Histone Modifications at Enhancers Reflect Global Cell Type-Specific Gene Expression.  Nature. 459(7243):108-12
</li></ul>
<ul><li>Visel, A, Blow, MJ, Li, Z, Zhang, T, Akiyama, JA, Holt, A, Plajzer-Frick, I, Shoukry, M, Wright, C, Chen, F, Afzal, V, <B>Ren, B</B>, Rubin, EM, Pennacchio, LA. (2009) ChIP-seq Accurately Predicts Tissue-Specific Activity of Enhancers.  Nature 457(7231): 854-8
</li></ul>
<ul><li>Won KJ, Agarwal S, Shen L, Shoemaker R, <B>Ren B</B>*, Wang W*. An integrated approach to identifying cis-regulatory modules in the human genome. (2009) PLoS ONE. 4(5): e5501. (co-corresponding authors)
</li></ul>
<ul><li>GC Hon, W Wang*, and <B>B Ren</B>* (2009) Discovery and Annotation of Functional Chromatin Signatures in the Human Genome. PLoS Comput Biol 5(11): e1000566. (*co-corresponding authors) 
</li></ul>
<ul><li>Lister R, Pelizzola M, Dowen RH, Hawkins RD, Hon G, Tonti-Filippini J, Nery JR, Lee L, Ye Z, Ngo QM, Edsall L, Antosiewicz-Bourget J, Stewart R, Ruotti V, Millar AH, Thomson JA, <B>Ren B</B>, Ecker JR. (2009) Human DNA methylomes at base resolution show widespread epigenomic differences. Nature. 462(7271):315-22
</li></ul>
<p><h3>2008</h3></p>
<ul><li>Barrera LO, Li Z, Smith AD, Zhang MQ, Green RD and <B>Ren B</B> (2008) Genome-wide mapping of active promoters in mouse embryonic stem cells and adult organs. Genome Research, 18(1):46-5
</li></ul>
<ul><li>Maynard, ND, Chen, Stuart, RK, Fan, JB, and <B>Ren, B</B>, (2008) Genome-wide Mapping of Allele-specific Protein-DNA Interactions in Human Cells.  Nature Methods, 5(4):307-9
</li></ul>
<ul><li>Johnson DS, Li W, Gordon DB, Bhattacharjee A, Curry B, Ghosh J, Brizuela L, Carroll JS, Brown M, Flicek P, Koch CM, Dunham I, Bieda M, Xu X, Farnham PJ, Kapranov P, Nix DA, Gingeras TR, Zhang X, Holster H, Jiang N, Green RD, Song JS, McCuine SA, Anton E, Nguyen L, Trinklein ND, Ye Z, Ching K, Hawkins D, <B>Ren B</B>, Scacheri PC, Rozowsky J, Karpikov A, Euskirchen G, Weissman S, Gerstein M, Snyder M, Yang A, Moqtaderi Z, Hirsch H, Shulha HP, Fu Y, Weng Z, Struhl K, Myers RM, Lieb JD, Liu XS. (2008) Systematic evaluation of variability in ChIP-chip experiments using predefined DNA targets. Genome Res. 18(3):393-403
</li></ul>
<ul><li>Li Y, Reddy MA, Miao F, Shanmugam N, Yee JK, Hawkins D, <B>Ren B</B>, Natarajan R. (2008) Role of the histone H3 lysine 4 methyltransferase, SET7/9, in the regulation of NF-kappa B dependent inflammatory genes: Relevance to diabetes and inflammation. J Biol Chem. 83(39):26771-81
</li></ul>
<ul><li>Hon G, <B>Ren B</B>*, Wang W* (2008) ChromaSig: A Probabilistic Approach to Finding Common Chromatin Signatures in the Human Genome. PLoS Comput Biol 4(10): e1000201. Epub 2008 Oct 17.  (co-corresponding authors) 
</li></ul>
<ul><li>25.	Won, KJ, Chepelev, I, <B>Ren, B</B> and Wang, W. (2008) Prediction of Regulatory Elements in Mammalian Genomes Using Chromatin Signatures. BMC Bioinformatics 9(1):547. 2008 [Epub ahead of print]
</li></ul>
<p><h3>2007</h3></p>
<ul><li>Zheng, M, Barrera, LO, <B>Ren, B</B> and Wu, Y (2007) ChIP-chip: data, model, and analysis. Biometrics, 63(3):787-96
</li></ul>
<ul><li>Heintzman N.D., Stuart R.K., Hon G., Fu Y., Barrera L.O., Van Calcar S., Qu C., Ching K.A., Wang W., Weng Z., Green R.D., Crawford G. and <B>Ren B</B>., (2007)  Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome.  Nature Genetics, 39:311-318
</li></ul>
<ul><li>Kim T.H., Abdullayev Z., Smith A., Ching K.A., Loukinov D., Green R.D., Zhang M.Q., Lobanenkov V., and <B>Ren B</B>. (2007) Analysis of the vertebrate insulator protein CTCF binding in the human genome. Cell, 128:1231-1245
</li></ul>
<ul><li>The ENCODE consortium (2007) The ENCODE pilot project: Identification and analysis of functional elements in 1% of the human genome. Nature 447:799-816
</li></ul>
<ul><li>E. Spiteri, G. Konopka, G. Coppola, J. Bomar, M. Oldham, J. Ou, S. C. Vernes, S. E. Fisher, <B>B. Ren</B>, D. H. Geschwind (2007) Identification of the transcriptional targets of FOXP2, a gene linked to speech and language in developing human brain. American J. of Human Genetics, 81:1144-1157
</li></ul>
<ul><li>Xi H, Shulha HP, Lin JM, Vales TR, Fu Y, Bodine DM, McKay RD, Chenoweth JG, Tesar PJ, Furey TS, <B>Ren B</B>, Weng Z, Crawford GE. (2007) Identification and Characterization of Cell Type-Specific and Ubiquitous Chromatin Regulatory Structures in the Human Genome. PLoS Genet. 3(8): e136
</li></ul>
<p><h3>2006</h3></p>
<ul><li>Sun, P, Xiong, H, Kim, TH, <B>Ren, B</B>, and Zhang Z (2006). Positive inter-regulation between beta-catenin/T cell factor-4 signaling and endothelin-1 signaling potentiates proliferation and survival of prostate cancer cells. Mol Pharmacol, 69:520-31
</li></ul>
<ul><li>Toyo-oka, K., Bowen T.J., Hirotsune S., Li, Z., Jain, S., Ota, S., Lozach, L.E., Bassett, I.G., Lozach, J., Rosenfeld, M.G., Glass, C.K., Eisenman, R., <B>Ren, B</B>., Hurlin, P. and Wynshaw-Boris, A., (2006) Mnt-deficient mammary glands exhibit impaired involution and tumors with characteristics of Myc overexpression.  Cancer Research, 66:5565-73
</li></ul>
<p><h3>2005</h3></p>
<ul><li>Kim, T.H., Xiong, H., Zhang, Z.H.* and <B>Ren, B</B>.* (2005)  β-catenin activates endothelin-1 in colon cancer cells.  Oncogene; 24:597-604. (co-corresponding authors) 
</li></ul>
<ul><li>Kim, TH, Barrera, LO, Qu, CX, Van Calcar, S, Trinklein, ND, Cooper, SJ, Luna, R, Glass, CK, Rosenfeld, MG, Myers, RM and <B>Ren, B</B> (2005) Direct isolation and identification of promoters in the human genome.  Genome Research; 15:830-839
</li></ul>
<ul><li>Kim, TH, Barrera, LO, Zheng, M, Qu, C, Singer, MA, Richmand, TA, Wu, Y, Green, RD and <B>Ren, B</B>. (2005) A high-resolution map of active promoters in the human genome.  Nature, 436:876-80
</li></ul>
<p><h3>2003</h3></p>
<ul><li>Li, Z., Van Calcar, S., Qu, C., Cavenee, W. K., Zhang, M., and <B>Ren, B</B>. (2003) A Global Regulatory Role for c-myc in Burkitt's Lymphoma Cells, Proc Natl Acad Sci U S A; 100:8164-8169. 
</li></ul>
