<!doctype html>
<html class="no-js" lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title></title>

    <link rel="stylesheet" type="text/css" href="/assets/css/styles_feeling_responsive.css">

  

	<script src="/assets/js/modernizr.min.js"></script>

	<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
	<script>
		WebFont.load({
			google: {
				families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
			}
		});
	</script>

	<noscript>
		<link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic%7CVolkhov' rel='stylesheet' type='text/css'>
	</noscript>


	<!-- Search Engine Optimization -->
	<meta name="description" content="»Feeling Responsive« is a responsive theme for Jekyll based on the fabulous foundation framework with beautiful typography and a bright color palette.">
	<meta name="google-site-verification" content="Vk0IOJ2jwG_qEoG7fuEXYqv0m2rLa8P778Fi_GrsgEQ">
	<meta name="msvalidate.01" content="0FB4C028ABCF07C908C54386ABD2D97F" >
	
	<link rel="author" href="https://plus.google.com/u/0/118311555303973066167">
	
	
	<link rel="canonical" href="//publications/peer-reviewed/">


	<!-- Facebook Open Graph -->
	<meta property="og:title" content="">
	<meta property="og:description" content="»Feeling Responsive« is a responsive theme for Jekyll based on the fabulous foundation framework with beautiful typography and a bright color palette.">
	<meta property="og:url" content="//publications/peer-reviewed/">
	<meta property="og:locale" content="en_EN">
	<meta property="og:type" content="website">
	<meta property="og:site_name" content="RENLAB">
	
	<meta property="article:author" content="https://www.facebook.com/phlow.media">


	
	<!-- Twitter -->
	<meta name="twitter:card" content="summary">
	<meta name="twitter:site" content="phlow">
	<meta name="twitter:creator" content="phlow">
	<meta name="twitter:title" content="">
	<meta name="twitter:description" content="»Feeling Responsive« is a responsive theme for Jekyll based on the fabulous foundation framework with beautiful typography and a bright color palette.">
	
	

	<link type="text/plain" rel="author" href="//humans.txt">

	

	

	<link rel="icon" sizes="32x32" href="/assets/img/favicon-32x32.png">

	<link rel="icon" sizes="192x192" href="/assets/img/touch-icon-192x192.png">

	<link rel="apple-touch-icon-precomposed" sizes="180x180" href="/assets/img/apple-touch-icon-180x180-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="152x152" href="/assets/img/apple-touch-icon-152x152-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="144x144" href="/assets/img/apple-touch-icon-144x144-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="120x120" href="/assets/img/apple-touch-icon-120x120-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="114x114" href="/assets/img/apple-touch-icon-114x114-precomposed.png">

	
	<link rel="apple-touch-icon-precomposed" sizes="76x76" href="/assets/img/apple-touch-icon-76x76-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="72x72" href="/assets/img/apple-touch-icon-72x72-precomposed.png">

	<link rel="apple-touch-icon-precomposed" href="/assets/img/apple-touch-icon-precomposed.png">	

	<meta name="msapplication-TileImage" content="//assets/img/msapplication_tileimage.png">

	<meta name="msapplication-TileColor" content="#fabb00">


	

</head>
<body id="top-of-page" class="post">
	
	<div id="navigation" class="sticky">
  <nav class="top-bar" role="navigation" data-topbar>
    <ul class="title-area">
      <li class="name">
      <h1 class="show-for-small-only"><a href="/" class="icon-tree"> RENLAB</a></h1>
    </li>
       <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->
      <li class="toggle-topbar menu-icon"><a href="#"><span>Nav</span></a></li>
    </ul>
    <section class="top-bar-section">

      <ul class="right">
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
            
            
              <li class="divider"></li>
              <li><a  href="//search/">Search</a></li>

            
            
          
        

              

          
          
            
            
              <li class="divider"></li>
              <li><a  href="//contact/">Contact</a></li>

            
            
          
        
        
      </ul>

      <ul class="left">
        

              

          
          

            
            
              <li><a  href="//">Home</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="//news/">News</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="//research/">Research</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="//team/">Team</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="//publication/">Publication</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="//publication/preprint/">Preprint</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          

            
            
              <li><a  href="//download/">Download</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          
        

              

          
          
        
        
      </ul>
    </section>
  </nav>
</div><!-- /#navigation -->

	

	

<div id="masthead-no-image-header">
	<div class="row">
		<div class="small-12 columns">
			<a id="logo" href="//" title="RENLAB – Laboratory of Gene Regulation">
				<img src="/assets/img/logo.png" alt="RENLAB – Laboratory of Gene Regulation">
			</a>
		</div><!-- /.small-12.columns -->
	</div><!-- /.row -->
</div><!-- /#masthead -->









	<div class="row t30">
	<div class="medium-8 columns medium-offset-2 end">
		<article itemscope itemtype="http://schema.org/Article">
			<header>
				

				<div itemprop="name">
					
					<h1>Peer Reviewed</h1>
				</div>
			</header>


			

			<div itemprop="articleSection">
			Note: *equal contribution. <sup>#</sup>co-correspondence. 


<ul> 


  
  
   </ul>
     <h3> 2023 </h3>
     <ul>
     
     
    <li>
      <i><b>A spatial genome aligner for resolving chromatin architectures from multiplexed DNA FISH </b> </i>
     <br> Bojing Blair Jia<sup>, Adam Jussila, Colin Kern, Quan Zhu, Bing Ren <br>
      <b> Nature Biotechnology</b>, 2023;doi:<a href="https://doi.org/10.1038/s41587-022-01568-9">10.1038/s41587-022-01568-9</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2022 </h3>
     <ul>
     
     
    <li>
      <i><b> Single-cell Epigenome Analysis Reveals Age-associated Decay of Heterochromatin Domains in Excitatory Neurons in the Mouse Brain </b> </i>
     <br>  Yanxiao Zhang*<sup>,#</sup>, Maria Luisa Amaral*, Chenxu Zhu, Steven F. Grieco, Xiaomeng Hou, Lin Lin, Justin Buchanan, Liqi Tong, Sebastian Preissl, Xiangmin Xu<sup>#</sup>, Bing Ren<sup>#</sup> <br>
      <b>  Cell Research</b>, 2022;doi:<a href="https://doi.org/10.1038/s41422-022-00719-6">10.1038/s41422-022-00719-6</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2021 </h3>
     <ul>
     
     
    <li>
      <i><b>A single-cell atlas of chromatin accessibility in the human genome </b> </i>
     <br> Kai Zhang*, James D. Hocker*, Michael Miller, Xiaomeng Hou, Joshua Chiou, Olivier B. Poirion, Yunjiang Qiu, Yang E. Li, Kyle J. Gaulton, Allen Wang, Sebastian Preissl, Bing Ren <br>
      <b> Cell</b>, 2021;doi:<a href="https://doi.org/https://doi.org/10.1016/j.cell.2021.10.024">https://doi.org/10.1016/j.cell.2021.10.024</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b>An Atlas of Gene Regulatory Elements in Adult Mouse Cerebrum </b> </i>
     <br> Yang Eric Li*, Sebastian Preissl*, Xiaomeng Hou, Ziyang Zhang, Kai Zhang, Yunjiang Qiu, Olivier Poirion, Bin Li, Joshua Chiou, Hanqing Liu, Antonio Pinto-Duarte, Naoki Kubo, Xiaoyu Yang, Rongxin Fang, Xinxin Wang, Jee Yun Han, Jacinta Lucero, Yiming Yan, Michael Miller, Samantha Kuan, David Gorkin, Kyle Gaulton, Yin Shen, Michael Nunn, Eran A. Mukame, M. Margarita Behrens, Joseph Ecker and Bing Ren <br>
      <b> Nature</b>, 2021;doi:<a href="https://doi.org/https://doi.org/10.1038/s41586-021-03604-1">https://doi.org/10.1038/s41586-021-03604-1</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b>SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data. </b> </i>
     <br>  Miao Yu*, Armen Abnousi*, Yanxiao Zhang, Guoqiang Li, Lindsay Lee, Ziyin Chen, Rongxin Fang, Taylor M. Lagler, Yuchen Yang, Jia Wen, Quan Sun, Yun Li, Bing Ren<sup>#</sup> & Ming Hu<sup>#</sup> <br>
      <b>  Nature Methods</b>, 2021;doi:<a href="https://doi.org/10.1038/s41592-021-01231-2">10.1038/s41592-021-01231-2</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b>CTCF Mediates Dosage and Sequence-context-dependent Transcriptional Insulation through Formation of Local Chromatin Domains </b> </i>
     <br> Hui Huang, Quan Zhu, Adam Jussila, Yuanyuan Han, Bogdan Bintu, Colin Kern, Mattia Conte, Yanxiao Zhang, Simona Bianco, Andrea Chiariello, Miao Yu, Rong Hu, Melodi Tastemel, Ivan Juric, Ming Hu, Mario Nicodemi, Xiaowei Zhuang, Bing Ren <br>
      <b> Nature Genetics</b>, 2021;doi:<a href="https://doi.org/https://doi.org/10.1038/s41588-021-00863-6">https://doi.org/10.1038/s41588-021-00863-6</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b>Cardiac Cell Type-Specific Gene Regulatory Programs and Disease Risk Association </b> </i>
     <br> James D Hocker,  Olivier B Poirion,  Fugui Zhu,  Justin Buchanan,  Kai Zhang,  Joshua Chiou,  Tsui-Min Wang, Qingquan Zhang,  Xiaomeng Hou,  Yang Eric Li,  Yanxiao Zhang,  Elie Farah,  Allen Wang,  Andrew D McCulloch,  Kyle J Gaulton,  Bing Ren<sup>#</sup>,  Neil C Chi<sup>#</sup>,  Sebastian Preissl<sup>#</sup> <br>
      <b> Science Advances</b>, 2021;doi:<a href="https://doi.org/10.1126/sciadv.abf1444">10.1126/sciadv.abf1444</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b>Comprehensive analysis of single cell ATAC-seq data with SnapATAC </b> </i>
     <br> Rongxin Fang, Sebastian Preissl, Yang Li, Xiaomeng Hou, Jacinta Lucero, Xinxin Wang, Amir Motamedi, Andrew K. Shiau, Xinzhu Zhou, Fangming Xie, Eran A. Mukamel, Kai Zhang, Yanxiao Zhang, M. Margarita Behrens, Joseph R. Ecker & Bing Ren <br>
      <b> Nature Communications</b>, 2021;doi:<a href="https://doi.org/https://doi.org/10.1038/s41467-021-21583-9">https://doi.org/10.1038/s41467-021-21583-9</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Joint profiling of histone modifications and transcriptome in single cells from mouse brain </b> </i>
     <br>  Chenxu Zhu, Yanxiao Zhang, Yang Eric Li, Jacinta Lucero, M. Margarita Behrens & Bing Ren <br>
      <b> Nature Methods</b>, 2021;doi:<a href="https://doi.org/https://doi.org/10.1038/s41592-021-01060-3">https://doi.org/10.1038/s41592-021-01060-3</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b>Systematic analysis of binding of transcription factors to noncoding variants </b> </i>
     <br> Jian Yan*<sup>,#</sup>, Yunjiang Qiu*, André M. Ribeiro dos Santos*, Yimeng Yin, Yang E. Li, Nick Vinckier, Naoki Nariai, Paola Benaglio, Anugraha Raman, Xiaoyu Li, Shicai Fan, Joshua Chiou, Fulin Chen, Kelly A. Frazer, Kyle J. Gaulton, Maike Sander, Jussi Taipale<sup>#</sup> & Bing Ren<sup>#</sup>  <br>
      <b> Nature</b>, 2021;doi:<a href="https://doi.org/https://doi.org/10.1038/s41586-021-03211-0">https://doi.org/10.1038/s41586-021-03211-0</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b>Promoter-proximal CTCF binding promotes distal enhancer-dependent gene activation </b> </i>
     <br> Naoki Kubo, Haruhiko Ishii, Xiong Xiong, Simona Bianco, Franz Meitinger, Rong Hu, James D. Hocker, Mattia Conte, David Gorkin, Miao Yu, Bin Li, Jesse R. Dixon, Ming Hu, Mario Nicodemi, Huimin Zhao & Bing Ren <br>
      <b> Nature Structural & Molecular Biology</b>, 2021;doi:<a href="https://doi.org/https://doi.org/10.1038/s41594-020-00539-5">https://doi.org/10.1038/s41594-020-00539-5</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2020 </h3>
     <ul>
     
     
    <li>
      <i><b>An atlas of dynamic chromatin landscapes in mouse fetal development </b> </i>
     <br> David U Gorkin*, Iros Barozzi*, Yuan Zhao*, Yanxiao Zhang*, Hui Huang*, Ah Young Lee, Bin Li, Joshua Chiou, Andre Wildberg, Bo Ding, Bo Zhang, Mengchi Wang, J Seth Strattan, Jean M Davidson, Yunjiang Qiu, Veena Afzal, Jennifer A Akiyama, Ingrid Plajzer-Frick, Catherine S Novak, Momoe Kato, Tyler H Garvin, Quan T Pham, Anne N Harrington, Brandon J Mannion, Elizabeth A Lee, Yoko Fukuda-Yuzawa, Yupeng He, Sebastian Preissl, Sora Chee, Jee Yun Han, Brian A Williams, Diane Trout, Henry Amrhein, Hongbo Yang, J Michael Cherry, Wei Wang, Kyle Gaulton, Joseph R Ecker, Yin Shen, Diane E Dickel, Axel Visel<sup>#</sup>, Len A Pennacchio<sup>#</sup>, Bing Ren<sup>#</sup> <br>
      <b> Nature</b>, 2020;doi:<a href="https://doi.org/https://doi.org/10.1038/s41586-020-2093-3">https://doi.org/10.1038/s41586-020-2093-3</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2019 </h3>
     <ul>
     
     
    <li>
      <i><b>Common DNA sequence variation influences 3-dimensional conformation of the human genome. </b> </i>
     <br> David U. Gorkin*, Yunjiang Qiu*, Ming Hu*<sup>,#</sup>, Kipper Fletez-Brant, Tristin Liu, Anthony D. Schmitt, Amina Noor, Joshua Chiou, Kyle J. Gaulton, Jonathan Sebat, Yun Li, Kasper D. Hansen & Bing Ren<sup>#</sup> <br>
      <b> Genome Biology</b>, 2019;doi:<a href="https://doi.org/https://doi.org/10.1186/s13059-019-1855-4">https://doi.org/10.1186/s13059-019-1855-4</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b>An ultra high-throughput method for single-cell joint analysis of open chromatin and transcriptome </b> </i>
     <br> Chenxu Zhu, Miao Yu, Hui Huang, Ivan Juric, Armen Abnousi, Rong Hu, Jacinta Lucero, M. Margarita Behrens, Ming Hu, Bing Ren  <br>
      <b>  Nature Structural & Molecular Biology</b>, 2019;doi:<a href="https://doi.org/https://doi.org/10.1038/s41594-019-0323-x">https://doi.org/10.1038/s41594-019-0323-x</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> A compendium of promoter-centered long-range chromatin interactions in the human genome </b> </i>
     <br> Inkyung Jung*<sup>,#</sup>, Anthony Schmitt*, Yarui Diao*, Andrew J. Lee, Tristin Liu, Dongchan Yang, Catherine Tan, Junghyun Eom, Marilynn Chan, Sora Chee, Zachary Chiang, Changyoun Kim, Eliezer Masliah, Cathy L. Barr, Bin Li, Samantha Kuan, Dongsup Kim,  Bing Ren<sup>#</sup> <br>
      <b>  Nature Genetics</b>, 2019;doi:<a href="https://doi.org/10.1038/s41588-019-0494-8">10.1038/s41588-019-0494-8</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Transcriptionally active HERV-H retrotransposons demarcate topologically associating domains in human pluripotent stem cells </b> </i>
     <br>  Yanxiao Zhang*, Ting Li*, Sebastian Preissl*, Maria Luisa Amaral, Jonathan D. Grinstein, Elie N. Farah, Eugin Destici, Yunjiang Qiu, Rong Hu, Ah Young Lee, Sora Chee, Kaiyue Ma, Zhen Ye, Quan Zhu, Hui Huang, Rongxin Fang, Leqian Yu, Juan Carlos Izpisua Belmonte, Jun Wu, Sylvia M. Evans, Neil C. Chi<sup>#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  Nature Genetics</b>, 2019;doi:<a href="https://doi.org/10.1038/s41588-019-0479-7">10.1038/s41588-019-0479-7</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Joint profiling of DNA methylation and chromatin architecture in single cells </b> </i>
     <br>  Guoqiang Li*, Yaping Liu*, Yanxiao Zhang, Naoki Kubo, Miao Yu, Rongxin Fang, Manolis Kellis,  Bing Ren <br>
      <b>  Nature Methods</b>, 2019;doi:<a href="https://doi.org/10.1038/s41592-019-0502-z">10.1038/s41592-019-0502-z</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> MAPS: Model-based analysis of long-range chromatin interactions from PLAC-seq and HiChIP experiments </b> </i>
     <br>  Ivan Juric*, Miao Yu*, Armen Abnousi, Ramya Raviram, Rongxin Fang, Yuan Zhao, Yanxiao Zhang, Yunjiang Qiu, Yuchen Yang, Yun Li, Bing Ren<sup>#</sup>,  Ming Hu<sup>#</sup> <br>
      <b>  PLoS computational biology</b>, 2019;doi:<a href="https://doi.org/10.1371/journal.pcbi.1006982">10.1371/journal.pcbi.1006982</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2018 </h3>
     <ul>
     
     
    <li>
      <i><b> Single-nucleus analysis of accessible chromatin in developing mouse forebrain reveals cell-type-specific transcriptional regulation </b> </i>
     <br>  Sebastian Preissl*, Rongxin Fang*, Hui Huang, Yuan Zhao, Ramya Raviram, David U. Gorkin, Yanxiao Zhang, Brandon C. Sos, Veena Afzal, Diane E. Dickel, Samantha Kuan, Axel Visel, Len A. Pennacchio, Kun Zhang,  Bing Ren <br>
      <b>  Nature Neuroscience</b>, 2018;doi:<a href="https://doi.org/10.1038/s41593-018-0079-3">10.1038/s41593-018-0079-3</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Histone H3 lysine 4 monomethylation modulates long-range chromatin interactions at enhancers </b> </i>
     <br>  Jian Yan*, Shi-An A. Chen*, Andrea Local, Tristin Liu, Yunjiang Qiu, Kristel M. Dorighi, Sebastian Preissl, Chloe M. Rivera, Chaochen Wang, Zhen Ye, Kai Ge, Ming Hu, Joanna Wysocka,  Bing Ren <br>
      <b>  Cell Research</b>, 2018;doi:<a href="https://doi.org/10.1038/cr.2018.1">10.1038/cr.2018.1</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Identification of H3K4me1-associated proteins at mammalian enhancers </b> </i>
     <br>  Andrea Local*, Hui Huang*, Claudio P. Albuquerque, Namit Singh, Ah Young Lee, Wei Wang, Chaochen Wang, Judy E. Hsia, Andrew K. Shiau, Kai Ge, Kevin D. Corbett, Dong Wang, Huilin Zhou,  Bing Ren <br>
      <b>  Nature Genetics</b>, 2018;doi:<a href="https://doi.org/10.1038/s41588-017-0015-6">10.1038/s41588-017-0015-6</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2017 </h3>
     <ul>
     
     
    <li>
      <i><b> The Three-Dimensional Organization of Mammalian Genomes </b> </i>
     <br>  Miao Yu,  Bing Ren <br>
      <b>  Annual Review of Cell and Developmental Biology</b>, 2017;doi:<a href="https://doi.org/10.1146/annurev-cellbio-100616-060531">10.1146/annurev-cellbio-100616-060531</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> A tiling-deletion-based genetic screen for cis-regulatory element identification in mammalian cells </b> </i>
     <br>  Yarui Diao*, Rongxin Fang*, Bin Li*, Zhipeng Meng, Juntao Yu, Yunjiang Qiu, Kimberly C. Lin, Hui Huang, Tristin Liu, Ryan J. Marina, Inkyung Jung, Yin Shen, Kun-Liang Guan,  Bing Ren <br>
      <b>  Nature Methods</b>, 2017;doi:<a href="https://doi.org/10.1038/nmeth.4264">10.1038/nmeth.4264</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> A Scalable Epitope Tagging Approach for High Throughput ChIP-Seq Analysis. </b> </i>
     <br>  Xiong Xiong, Yanxiao Zhang, Jian Yan, Surbhi Jain, Sora Chee, Bing Ren<sup>#</sup>,  Huimin Zhao<sup>#</sup> <br>
      <b>  ACS synthetic biology</b>, 2017;doi:<a href="https://doi.org/10.1021/acssynbio.6b00358">10.1021/acssynbio.6b00358</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2016 </h3>
     <ul>
     
     
    <li>
      <i><b> Mapping of long-range chromatin interactions by proximity ligation-assisted ChIP-seq. </b> </i>
     <br>  Rongxin Fang*, Miao Yu*, Guoqiang Li, Sora Chee, Tristin Liu, Anthony D. Schmitt,  Bing Ren <br>
      <b>  Cell research</b>, 2016;doi:<a href="https://doi.org/10.1038/cr.2016.137">10.1038/cr.2016.137</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> A Compendium of Chromatin Contact Maps Reveals Spatially Active Regions in the Human Genome. </b> </i>
     <br>  Anthony D. Schmitt*, Ming Hu*<sup>,#</sup>, Inkyung Jung, Zheng Xu, Yunjiang Qiu, Catherine L. Tan, Yun Li, Shin Lin, Yiing Lin, Cathy L. Barr,  Bing Ren<sup>#</sup> <br>
      <b>  Cell reports</b>, 2016;doi:<a href="https://doi.org/10.1016/j.celrep.2016.10.061">10.1016/j.celrep.2016.10.061</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Broad histone H3K4me3 domains in mouse oocytes modulate maternal-to-zygotic transition. </b> </i>
     <br>  John Arne Dahl*<sup>,#</sup>, Inkyung Jung*, Havard Aanes, Gareth D. Greggains, Adeel Manaf, Mads Lerdrup, Guoqiang Li, Samantha Kuan, Bin Li, Ah Young Lee, Sebastian Preissl, Ingunn Jermstad, Mads Haugland Haugen, Rajikala Suganthan, Magnar Bjoras, Klaus Hansen, Knut Tomas Dalen, Peter Fedorcsak, Bing Ren<sup>#</sup>, Arne Klungland<sup>#</sup> <br>
      <b>  Nature</b>, 2016;doi:<a href="https://doi.org/10.1038/nature19360">10.1038/nature19360</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Chromatin Domains: The Unit of Chromosome Organization. </b> </i>
     <br>  Jesse R. Dixon, David U. Gorkin,  Bing Ren <br>
      <b>  Molecular cell</b>, 2016;doi:<a href="https://doi.org/10.1016/j.molcel.2016.05.018">10.1016/j.molcel.2016.05.018</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> A new class of temporarily phenotypic enhancers identified by CRISPR/Cas9-mediated genetic screening. </b> </i>
     <br>  Yarui Diao, Bin Li, Zhipeng Meng, Inkyung Jung, Ah Young Lee, Jesse Dixon, Lenka Maliskova, Kun-Liang Guan, Yin Shen<sup>#</sup>, Bing Ren<sup>#</sup> <br>
      <b>  Genome research</b>, 2016;doi:<a href="https://doi.org/10.1101/gr.197152.115">10.1101/gr.197152.115</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2015 </h3>
     <ul>
     
     
    <li>
      <i><b> Complete haplotype phasing of the MHC and KIR loci with targeted HaploSeq. </b> </i>
     <br>  Siddarth Selvaraj*, Anthony D. Schmitt*, Jesse R. Dixon,  Bing Ren <br>
      <b>  BMC genomics</b>, 2015;doi:<a href="https://doi.org/10.1186/s12864-015-1949-7">10.1186/s12864-015-1949-7</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> EGFR Mutation Promotes Glioblastoma through Epigenome and Transcription Factor Network Remodeling. </b> </i>
     <br>  Feng Liu*, Gary C. Hon*, Genaro R. Villa, Kristen M. Turner, Shiro Ikegami, Huijun Yang, Zhen Ye, Bin Li, Samantha Kuan, Ah Young Lee, Ciro Zanca, Bowen Wei, Greg Lucey, David Jenkins, Wei Zhang, Cathy L. Barr, Frank B. Furnari, Timothy F. Cloughesy, William H. Yong, Timothy C. Gahman, Andrew K. Shiau, Webster K. Cavenee, Bing Ren<sup>#</sup>,  Paul S. Mischel<sup>#</sup> <br>
      <b>  Molecular cell</b>, 2015;doi:<a href="https://doi.org/10.1016/j.molcel.2015.09.002">10.1016/j.molcel.2015.09.002</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Comparative analyses of CTCF and BORIS occupancies uncover two distinct classes of CTCF binding genomic regions. </b> </i>
     <br>  Elena M. Pugacheva, Samuel Rivero-Hinojosa, Celso A. Espinoza, Claudia Fabiola Mendez-Catala, Sungyun Kang, Teruhiko Suzuki, Natsuki Kosaka-Suzuki, Susan Robinson, Vijayaraj Nagarajan, Zhen Ye, Abdelhalim Boukaba, John E. J. Rasko, Alexander V. Strunnikov, Dmitri Loukinov, Bing Ren<sup>#</sup>,  Victor V. Lobanenkov<sup>#</sup> <br>
      <b>  Genome biology</b>, 2015;doi:<a href="https://doi.org/10.1186/s13059-015-0736-8">10.1186/s13059-015-0736-8</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> MPE-seq a new method for the genome-wide analysis of chromatin structure. </b> </i>
     <br>  Haruhiko Ishii, James T. Kadonaga<sup>#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  Proceedings of the National Academy of Sciences of the United States of America</b>, 2015;doi:<a href="https://doi.org/10.1073/pnas.1424804112">10.1073/pnas.1424804112</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> A CRISPR Connection between Chromatin Topology and Genetic Disorders. </b> </i>
     <br>  Bing Ren,  Jesse R. Dixon <br>
      <b>  Cell</b>, 2015;doi:<a href="https://doi.org/10.1016/j.cell.2015.04.047">10.1016/j.cell.2015.04.047</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Epigenetic priming of enhancers predicts developmental competence of hESC-derived endodermal lineage intermediates. </b> </i>
     <br>  Allen Wang*, Feng Yue*, Yan Li*, Ruiyu Xie, Thomas Harper, Nisha A. Patel, Kayla Muth, Jeffrey Palmer, Yunjiang Qiu, Jinzhao Wang, Dieter K. Lam, Jeffrey C. Raum, Doris A. Stoffers, Bing Ren<sup>#</sup>,  Maike Sander<sup>#</sup> <br>
      <b>  Cell stem cell</b>, 2015;doi:<a href="https://doi.org/10.1016/j.stem.2015.02.013">10.1016/j.stem.2015.02.013</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Integrative analysis of haplotype-resolved epigenomes across human tissues. </b> </i>
     <br>  Danny Leung*, Inkyung Jung*, Nisha Rajagopal*, Anthony Schmitt, Siddarth Selvaraj, Ah Young Lee, Chia-An Yen, Shin Lin, Yiing Lin, Yunjiang Qiu, Wei Xie, Feng Yue, Manoj Hariharan, Pradipta Ray, Samantha Kuan, Lee Edsall, Hongbo Yang, Neil C. Chi, Michael Q. Zhang, Joseph R. Ecker,  Bing Ren <br>
      <b>  Nature</b>, 2015;doi:<a href="https://doi.org/10.1038/nature14217">10.1038/nature14217</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Integrative analysis of 111 reference human epigenomes. </b> </i>
     <br>  Anshul Kundaje*, Wouter Meuleman*, Jason Ernst*, Misha Bilenky*, Angela Yen, Alireza Heravi-Moussavi, Pouya Kheradpour, Zhizhuo Zhang, Jianrong Wang, Michael J. Ziller, Viren Amin, John W. Whitaker, Matthew D. Schultz, Lucas D. Ward, Abhishek Sarkar, Gerald Quon, Richard S. Sandstrom, Matthew L. Eaton, Yi-Chieh Wu, Andreas R. Pfenning, Xinchen Wang, Melina Claussnitzer, Yaping Liu, Cristian Coarfa, R. Alan Harris, Noam Shoresh, Charles B. Epstein, Elizabeta Gjoneska, Danny Leung, Wei Xie, R. David Hawkins, Ryan Lister, Chibo Hong, Philippe Gascard, Andrew J. Mungall, Richard Moore, Eric Chuah, Angela Tam, Theresa K. Canfield, R. Scott Hansen, Rajinder Kaul, Peter J. Sabo, Mukul S. Bansal, Annaick Carles, Jesse R. Dixon, Kai-How Farh, Soheil Feizi, Rosa Karlic, Ah-Ram Kim, Ashwinikumar Kulkarni, Daofeng Li, Rebecca Lowdon, GiNell Elliott, Tim R. Mercer, Shane J. Neph, Vitor Onuchic, Paz Polak, Nisha Rajagopal, Pradipta Ray, Richard C. Sallari, Kyle T. Siebenthall, Nicholas A. Sinnott-Armstrong, Michael Stevens, Robert E. Thurman, Jie Wu, Bo Zhang, Xin Zhou, Arthur E. Beaudet, Laurie A. Boyer, Philip L. De Jager, Peggy J. Farnham, Susan J. Fisher, David Haussler, Steven J. M. Jones, Wei Li, Marco A. Marra, Michael T. McManus, Shamil Sunyaev, James A. Thomson, Thea D. Tlsty, Li-Huei Tsai, Wei Wang, Robert A. Waterland, Michael Q. Zhang, Lisa H. Chadwick, Bradley E. Bernstein<sup>#</sup>, Joseph F. Costello<sup>#</sup>, Joseph R. Ecker<sup>#</sup>, Martin Hirst<sup>#</sup>, Alexander Meissner<sup>#</sup>, Aleksandar Milosavljevic<sup>#</sup>, Bing Ren<sup>#</sup>, John A. Stamatoyannopoulos<sup>#</sup>, Ting Wang<sup>#</sup>,  Manolis Kellis<sup>#</sup> <br>
      <b>  Nature</b>, 2015;doi:<a href="https://doi.org/10.1038/nature14248">10.1038/nature14248</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Chromatin architecture reorganization during stem cell differentiation. </b> </i>
     <br>  Jesse R. Dixon*, Inkyung Jung*, Siddarth Selvaraj*, Yin Shen, Jessica E. Antosiewicz-Bourget, Ah Young Lee, Zhen Ye, Audrey Kim, Nisha Rajagopal, Wei Xie, Yarui Diao, Jing Liang, Huimin Zhao, Victor V. Lobanenkov, Joseph R. Ecker, James A. Thomson,  Bing Ren <br>
      <b>  Nature</b>, 2015;doi:<a href="https://doi.org/10.1038/nature14222">10.1038/nature14222</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Transcriptional Enhancers: Bridging the Genome and Phenome. </b> </i>
     <br>  Bing Ren,  Feng Yue <br>
      <b>  Cold Spring Harbor symposia on quantitative biology</b>, 2015;doi:<a href="https://doi.org/10.1101/sqb.2015.80.027219">10.1101/sqb.2015.80.027219</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2014 </h3>
     <ul>
     
     
    <li>
      <i><b> A comparative encyclopedia of DNA elements in the mouse genome. </b> </i>
     <br>  Feng Yue*, Yong Cheng*, Alessandra Breschi*, Jeff Vierstra*, Weisheng Wu*, Tyrone Ryba*, Richard Sandstrom*, Zhihai Ma*, Carrie Davis*, Benjamin D. Pope*, Yin Shen*, Dmitri D. Pervouchine, Sarah Djebali, Robert E. Thurman, Rajinder Kaul, Eric Rynes, Anthony Kirilusha, Georgi K. Marinov, Brian A. Williams, Diane Trout, Henry Amrhein, Katherine Fisher-Aylor, Igor Antoshechkin, Gilberto DeSalvo, Lei-Hoon See, Meagan Fastuca, Jorg Drenkow, Chris Zaleski, Alex Dobin, Pablo Prieto, Julien Lagarde, Giovanni Bussotti, Andrea Tanzer, Olgert Denas, Kanwei Li, M. A. Bender, Miaohua Zhang, Rachel Byron, Mark T. Groudine, David McCleary, Long Pham, Zhen Ye, Samantha Kuan, Lee Edsall, Yi-Chieh Wu, Matthew D. Rasmussen, Mukul S. Bansal, Manolis Kellis, Cheryl A. Keller, Christapher S. Morrissey, Tejaswini Mishra, Deepti Jain, Nergiz Dogan, Robert S. Harris, Philip Cayting, Trupti Kawli, Alan P. Boyle, Ghia Euskirchen, Anshul Kundaje, Shin Lin, Yiing Lin, Camden Jansen, Venkat S. Malladi, Melissa S. Cline, Drew T. Erickson, Vanessa M. Kirkup, Katrina Learned, Cricket A. Sloan, Kate R. Rosenbloom, Beatriz Lacerda de Sousa, Kathryn Beal, Miguel Pignatelli, Paul Flicek, Jin Lian, Tamer Kahveci, Dongwon Lee, W. James Kent, Miguel Ramalho Santos, Javier Herrero, Cedric Notredame, Audra Johnson, Shinny Vong, Kristen Lee, Daniel Bates, Fidencio Neri, Morgan Diegel, Theresa Canfield, Peter J. Sabo, Matthew S. Wilken, Thomas A. Reh, Erika Giste, Anthony Shafer, Tanya Kutyavin, Eric Haugen, Douglas Dunn, Alex P. Reynolds, Shane Neph, Richard Humbert, R. Scott Hansen, Marella De Bruijn, Licia Selleri, Alexander Rudensky, Steven Josefowicz, Robert Samstein, Evan E. Eichler, Stuart H. Orkin, Dana Levasseur, Thalia Papayannopoulou, Kai-Hsin Chang, Arthur Skoultchi, Srikanta Gosh, Christine Disteche, Piper Treuting, Yanli Wang, Mitchell J. Weiss, Gerd A. Blobel, Xiaoyi Cao, Sheng Zhong, Ting Wang, Peter J. Good, Rebecca F. Lowdon, Leslie B. Adams, Xiao-Qiao Zhou, Michael J. Pazin, Elise A. Feingold, Barbara Wold, James Taylor, Ali Mortazavi, Sherman M. Weissman, John A. Stamatoyannopoulos<sup>#</sup>, Michael P. Snyder<sup>#</sup>, Roderic Guigo<sup>#</sup>, Thomas R. Gingeras<sup>#</sup>, David M. Gilbert<sup>#</sup>, Ross C. Hardison<sup>#</sup>, Michael A. Beer*<sup>,#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  Nature</b>, 2014;doi:<a href="https://doi.org/10.1038/nature13992">10.1038/nature13992</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> 5mC oxidation by Tet2 modulates enhancer activity and timing of transcriptome reprogramming during differentiation. </b> </i>
     <br>  Gary C. Hon, Chun-Xiao Song, Tingting Du, Fulai Jin, Siddarth Selvaraj, Ah Young Lee, Chia-An Yen, Zhen Ye, Shi-Qing Mao, Bang-An Wang, Samantha Kuan, Lee E. Edsall, Boxuan Simen Zhao, Guo-Liang Xu, Chuan He<sup>#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  Molecular cell</b>, 2014;doi:<a href="https://doi.org/10.1016/j.molcel.2014.08.026">10.1016/j.molcel.2014.08.026</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Distinct and predictive histone lysine acetylation patterns at promoters enhancers and gene bodies. </b> </i>
     <br>  Nisha Rajagopal, Jason Ernst, Pradipta Ray, Jie Wu, Michael Zhang, Manolis Kellis,  Bing Ren <br>
      <b>  G3 (Bethesda Md.)</b>, 2014;doi:<a href="https://doi.org/10.1534/g3.114.013565">10.1534/g3.114.013565</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> The 3D genome in transcriptional regulation and pluripotency. </b> </i>
     <br>  David U. Gorkin, Danny Leung,  Bing Ren <br>
      <b>  Cell stem cell</b>, 2014;doi:<a href="https://doi.org/10.1016/j.stem.2014.05.017">10.1016/j.stem.2014.05.017</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Regulation of DNA methylation turnover at LTR retrotransposons and imprinted loci by the histone methyltransferase Setdb1. </b> </i>
     <br>  Danny Leung*, Tingting Du*, Ulrich Wagner*, Wei Xie, Ah Young Lee, Preeti Goyal, Yujing Li, Keith E. Szulwach, Peng Jin, Matthew C. Lorincz<sup>#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  Proceedings of the National Academy of Sciences of the United States of America</b>, 2014;doi:<a href="https://doi.org/10.1073/pnas.1322273111">10.1073/pnas.1322273111</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Cohesin and CTCF differentially affect chromatin architecture and gene expression in human cells. </b> </i>
     <br>  Jessica Zuin*, Jesse R. Dixon*, Michael I. J. A. van der Reijden, Zhen Ye, Petros Kolovos, Rutger W. W. Brouwer, Mariette P. C. van de Corput, Harmen J. G. van de Werken, Tobias A. Knoch, Wilfred F. J. van IJcken, Frank G. Grosveld, Bing Ren<sup>#</sup>,  Kerstin S. Wendt<sup>#</sup> <br>
      <b>  Proceedings of the National Academy of Sciences of the United States of America</b>, 2014;doi:<a href="https://doi.org/10.1073/pnas.1317788111">10.1073/pnas.1317788111</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> CRISPR reveals a distal super-enhancer required for Sox2 expression in mouse embryonic stem cells. </b> </i>
     <br>  Yan Li*, Chloe M. Rivera*, Haruhiko Ishii*, Fulai Jin, Siddarth Selvaraj, Ah Young Lee, Jesse R. Dixon,  Bing Ren <br>
      <b>  PloS one</b>, 2014;doi:<a href="https://doi.org/10.1371/journal.pone.0114485">10.1371/journal.pone.0114485</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2013 </h3>
     <ul>
     
     
    <li>
      <i><b> Whole-genome haplotype reconstruction using proximity-ligation and shotgun sequencing. </b> </i>
     <br>  Siddarth Selvaraj*, Jesse R Dixon*, Vikas Bansal,  Bing Ren <br>
      <b>  Nature biotechnology</b>, 2013;doi:<a href="https://doi.org/10.1038/nbt.2728">10.1038/nbt.2728</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> A high-resolution map of the three-dimensional chromatin interactome in human cells. </b> </i>
     <br>  Fulai Jin*, Yan Li*, Jesse R. Dixon, Siddarth Selvaraj, Zhen Ye, Ah Young Lee, Chia-An Yen, Anthony D. Schmitt, Celso A. Espinoza,  Bing Ren <br>
      <b>  Nature</b>, 2013;doi:<a href="https://doi.org/10.1038/nature12644">10.1038/nature12644</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Epigenetic memory at embryonic enhancers identified in DNA methylation maps from  adult mouse tissues. </b> </i>
     <br>  Gary C. Hon, Nisha Rajagopal, Yin Shen, David F. McCleary, Feng Yue, My D. Dang,  Bing Ren <br>
      <b>  Nature genetics</b>, 2013;doi:<a href="https://doi.org/10.1038/ng.2746">10.1038/ng.2746</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Mapping human epigenomes. </b> </i>
     <br>  Chloe M. Rivera,  Bing Ren <br>
      <b>  Cell</b>, 2013;doi:<a href="https://doi.org/10.1016/j.cell.2013.09.011">10.1016/j.cell.2013.09.011</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Developmental biology. Enhancing pluripotency and lineage specification. </b> </i>
     <br>  Wei Xie,  Bing Ren <br>
      <b>  Science (New York N.Y.)</b>, 2013;doi:<a href="https://doi.org/10.1126/science.1236254">10.1126/science.1236254</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Genome organization and long-range regulation of gene expression by enhancers. </b> </i>
     <br>  Andrea Smallwood,  Bing Ren <br>
      <b>  Current opinion in cell biology</b>, 2013;doi:<a href="https://doi.org/10.1016/j.ceb.2013.02.005">10.1016/j.ceb.2013.02.005</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Epigenomic analysis of multilineage differentiation of human embryonic stem cells. </b> </i>
     <br>  Wei Xie, Matthew D. Schultz, Ryan Lister, Zhonggang Hou, Nisha Rajagopal, Pradipta Ray, John W. Whitaker, Shulan Tian, R. David Hawkins, Danny Leung, Hongbo Yang, Tao Wang, Ah Young Lee, Scott A. Swanson, Jiuchun Zhang, Yun Zhu, Audrey Kim, Joseph R. Nery, Mark A. Urich, Samantha Kuan, Chia-an Yen, Sarit Klugman, Pengzhi Yu, Kran Suknuntha, Nicholas E. Propson, Huaming Chen, Lee E. Edsall, Ulrich Wagner, Yan Li, Zhen Ye, Ashwinikumar Kulkarni, Zhenyu Xuan, Wen-Yu Chung, Neil C. Chi, Jessica E. Antosiewicz-Bourget, Igor Slukvin, Ron Stewart, Michael Q. Zhang, Wei Wang, James A. Thomson<sup>#</sup>, Joseph R. Ecker<sup>#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  Cell</b>, 2013;doi:<a href="https://doi.org/10.1016/j.cell.2013.04.022">10.1016/j.cell.2013.04.022</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> RFECS: a random-forest based algorithm for enhancer identification from chromatin state. </b> </i>
     <br>  Nisha Rajagopal, Wei Xie, Yan Li, Uli Wagner, Wei Wang, John Stamatoyannopoulos, Jason Ernst, Manolis Kellis,  Bing Ren <br>
      <b>  PLoS computational biology</b>, 2013;doi:<a href="https://doi.org/10.1371/journal.pcbi.1002968">10.1371/journal.pcbi.1002968</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2012 </h3>
     <ul>
     
     
    <li>
      <i><b> A map of the cis-regulatory sequences in the mouse genome. </b> </i>
     <br>  Yin Shen*, Feng Yue*, David F. McCleary, Zhen Ye, Lee Edsall, Samantha Kuan, Ulrich Wagner, Jesse Dixon, Leonard Lee, Victor V. Lobanenkov,  Bing Ren <br>
      <b>  Nature</b>, 2012;doi:<a href="https://doi.org/10.1038/nature11243">10.1038/nature11243</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> CBX3 regulates efficient RNA processing genome-wide. </b> </i>
     <br>  Andrea Smallwood, Gary C. Hon, Fulai Jin, Ryan E. Henry, Joaquin M. Espinosa,  Bing Ren <br>
      <b>  Genome research</b>, 2012;doi:<a href="https://doi.org/10.1101/gr.124818.111">10.1101/gr.124818.111</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome. </b> </i>
     <br>  Miao Yu*, Gary C. Hon*, Keith E. Szulwach*, Chun-Xiao Song, Liang Zhang, Audrey Kim, Xuekun Li, Qing Dai, Yin Shen, Beomseok Park, Jung-Hyun Min, Peng Jin, Bing Ren<sup>#</sup>,  Chuan He<sup>#</sup> <br>
      <b>  Cell</b>, 2012;doi:<a href="https://doi.org/10.1016/j.cell.2012.04.027">10.1016/j.cell.2012.04.027</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Topological domains in mammalian genomes identified by analysis of chromatin interactions. </b> </i>
     <br>  Jesse R. Dixon, Siddarth Selvaraj, Feng Yue, Audrey Kim, Yan Li, Yin Shen, Ming Hu, Jun S. Liu,  Bing Ren <br>
      <b>  Nature</b>, 2012;doi:<a href="https://doi.org/10.1038/nature11082">10.1038/nature11082</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Base-resolution analyses of sequence and parent-of-origin dependent DNA methylation in the mouse genome. </b> </i>
     <br>  Wei Xie, Cathy L. Barr<sup>#</sup>, Audrey Kim, Feng Yue, Ah Young Lee, James Eubanks, Emma L. Dempster,  Bing Ren<sup>#</sup> <br>
      <b>  Cell</b>, 2012;doi:<a href="https://doi.org/10.1016/j.cell.2011.12.035">10.1016/j.cell.2011.12.035</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Global DNA hypomethylation coupled to repressive chromatin domain formation and gene silencing in breast cancer. </b> </i>
     <br>  Gary C. Hon, R. David Hawkins, Otavia L. Caballero, Christine Lo, Ryan Lister, Mattia Pelizzola, Armand Valsesia, Zhen Ye, Samantha Kuan, Lee E. Edsall, Anamaria Aranha Camargo, Brian J. Stevenson, Joseph R. Ecker, Vineet Bafna, Robert L. Strausberg, Andrew J. Simpson,  Bing Ren <br>
      <b>  Genome research</b>, 2012;doi:<a href="https://doi.org/10.1101/gr.125872.111">10.1101/gr.125872.111</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2011 </h3>
     <ul>
     
     
    <li>
      <i><b> Enhancers: multi-dimensional signal integrators. </b> </i>
     <br>  Fulai Jin, Yan Li, Bing Ren,  Rama Natarajan <br>
      <b>  Transcription</b>, 2011;doi:<a href="https://doi.org/10.4161/trns.2.5.17712">10.4161/trns.2.5.17712</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Dynamic chromatin states in human ES cells reveal potential regulatory sequences  and genes involved in pluripotency. </b> </i>
     <br>  R. David Hawkins*, Gary C. Hon*, Chuhu Yang, Jessica E. Antosiewicz-Bourget, Leonard K. Lee, Que-Minh Ngo, Sarit Klugman, Keith A. Ching, Lee E. Edsall, Zhen Ye, Samantha Kuan, Pengzhi Yu, Hui Liu, Xinmin Zhang, Roland D. Green, Victor V. Lobanenkov, Ron Stewart, James A. Thomson<sup>#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  Cell research</b>, 2011;doi:<a href="https://doi.org/10.1038/cr.2011.146">10.1038/cr.2011.146</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Mapping higher order structure of chromatin domains. </b> </i>
     <br>  Celso A. Espinoza,  Bing Ren <br>
      <b>  Nature genetics</b>, 2011;doi:<a href="https://doi.org/10.1038/ng.869">10.1038/ng.869</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> PU.1 and C/EBP(alpha) synergistically program distinct response to NF-kappaB activation through establishing monocyte specific enhancers. </b> </i>
     <br>  Fulai Jin*, Yan Li*, Bing Ren<sup>#</sup>,  Rama Natarajan<sup>#</sup> <br>
      <b>  Proceedings of the National Academy of Sciences of the United States of America</b>, 2011;doi:<a href="https://doi.org/10.1073/pnas.1017214108">10.1073/pnas.1017214108</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2010 </h3>
     <ul>
     
     
    <li>
      <i><b> Next-generation genomics: an integrative approach. </b> </i>
     <br>  R. David Hawkins, Gary C. Hon,  Bing Ren <br>
      <b>  Nature reviews. Genetics</b>, 2010;doi:<a href="https://doi.org/10.1038/nrg2795">10.1038/nrg2795</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Transcription: Enhancers make non-coding RNA. </b> </i>
     <br>   Bing Ren <br>
      <b>  Nature</b>, 2010;doi:<a href="https://doi.org/10.1038/465173a">10.1038/465173a</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Distinct epigenomic landscapes of pluripotent and lineage-committed human cells. </b> </i>
     <br>  R. David Hawkins*, Gary C. Hon*, Leonard K. Lee, Queminh Ngo, Ryan Lister, Mattia Pelizzola, Lee E. Edsall, Samantha Kuan, Ying Luu, Sarit Klugman, Jessica Antosiewicz-Bourget, Zhen Ye, Celso Espinoza, Saurabh Agarwahl, Li Shen, Victor Ruotti, Wei Wang, Ron Stewart, James A. Thomson, Joseph R. Ecker,  Bing Ren <br>
      <b>  Cell stem cell</b>, 2010;doi:<a href="https://doi.org/10.1016/j.stem.2010.03.018">10.1016/j.stem.2010.03.018</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2009 </h3>
     <ul>
     
     
    <li>
      <i><b> Finding distal regulatory elements in the human genome. </b> </i>
     <br>  Nathaniel D. Heintzman,  Bing Ren <br>
      <b>  Current opinion in genetics & development</b>, 2009;doi:<a href="https://doi.org/10.1016/j.gde.2009.09.006">10.1016/j.gde.2009.09.006</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Discovery and annotation of functional chromatin signatures in the human genome. </b> </i>
     <br>  Gary Hon, Wei Wang<sup>#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  PLoS computational biology</b>, 2009;doi:<a href="https://doi.org/10.1371/journal.pcbi.1000566">10.1371/journal.pcbi.1000566</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Predictive chromatin signatures in the mammalian genome. </b> </i>
     <br>  Gary C. Hon, R. David Hawkins,  Bing Ren <br>
      <b>  Human molecular genetics</b>, 2009;doi:<a href="https://doi.org/10.1093/hmg/ddp409">10.1093/hmg/ddp409</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Histone modifications at human enhancers reflect global cell-type-specific gene expression. </b> </i>
     <br>  Nathaniel D. Heintzman*, Gary C. Hon*, R. David Hawkins*, Pouya Kheradpour, Alexander Stark, Lindsey F. Harp, Zhen Ye, Leonard K. Lee, Rhona K. Stuart, Christina W. Ching, Keith A. Ching, Jessica E. Antosiewicz-Bourget, Hui Liu, Xinmin Zhang, Roland D. Green, Victor V. Lobanenkov, Ron Stewart, James A. Thomson, Gregory E. Crawford, Manolis Kellis,  Bing Ren <br>
      <b>  Nature</b>, 2009;doi:<a href="https://doi.org/10.1038/nature07829">10.1038/nature07829</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> An integrated approach to identifying cis-regulatory modules in the human genome. </b> </i>
     <br>  Kyoung-Jae Won, Saurabh Agarwal, Li Shen, Robert Shoemaker, Bing Ren<sup>#</sup>,  Wei Wang<sup>#</sup> <br>
      <b>  PloS one</b>, 2009;doi:<a href="https://doi.org/10.1371/journal.pone.0005501">10.1371/journal.pone.0005501</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2008 </h3>
     <ul>
     
     
    <li>
      <i><b> ChromaSig: a probabilistic approach to finding common chromatin signatures in the human genome. </b> </i>
     <br>  Gary Hon, Bing Ren<sup>#</sup>,  Wei Wang<sup>#</sup> <br>
      <b>  PLoS computational biology</b>, 2008;doi:<a href="https://doi.org/10.1371/journal.pcbi.1000201">10.1371/journal.pcbi.1000201</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Genome-wide mapping of allele-specific protein-DNA interactions in human cells. </b> </i>
     <br>  Nathaniel D. Maynard, Jing Chen, Rhona K. Stuart, Jian-Bing Fan,  Bing Ren <br>
      <b>  Nature methods</b>, 2008;doi:<a href="https://doi.org/10.1038/nmeth.1194">10.1038/nmeth.1194</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Genome-wide mapping and analysis of active promoters in mouse embryonic stem cells and adult organs. </b> </i>
     <br>  Leah O. Barrera*, Zirong Li*, Andrew D. Smith, Karen C. Arden, Webster K. Cavenee, Michael Q. Zhang, Roland D. Green,  Bing Ren <br>
      <b>  Genome research</b>, 2008;doi:<a href="https://doi.org/10.1101/gr.6654808">10.1101/gr.6654808</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2007 </h3>
     <ul>
     
     
    <li>
      <i><b> Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project </b> </i>
     <br>  The ENCODE Project Consortium <br>
      <b>  Nature</b>, 2007;doi:<a href="https://doi.org/10.1038/nature05874">10.1038/nature05874</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Analysis of the vertebrate insulator protein CTCF-binding sites in the human genome. </b> </i>
     <br>  Tae Hoon Kim<sup>#</sup>, Ziedulla K. Abdullaev, Andrew D. Smith, Keith A. Ching, Dmitri I. Loukinov, Roland D. Green, Michael Q. Zhang, Victor V. Lobanenkov,  Bing Ren<sup>#</sup> <br>
      <b>  Cell</b>, 2007;doi:<a href="https://doi.org/10.1016/j.cell.2006.12.048">10.1016/j.cell.2006.12.048</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome. </b> </i>
     <br>  Nathaniel D. Heintzman, Rhona K. Stuart, Gary Hon, Yutao Fu, Christina W. Ching, R. David Hawkins, Leah O. Barrera, Sara Van Calcar, Chunxu Qu, Keith A. Ching, Wei Wang, Zhiping Weng, Roland D. Green, Gregory E. Crawford,  Bing Ren <br>
      <b>  Nature genetics</b>, 2007;doi:<a href="https://doi.org/10.1038/ng1966">10.1038/ng1966</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> The gateway to transcription: identifying characterizing and understanding promoters in the eukaryotic genome </b> </i>
     <br>  N. D. Heintzman,  B. Ren <br>
      <b>  Cellular and molecular life sciences: CMLS</b>, 2007;doi:<a href="https://doi.org/10.1007/s00018-006-6295-0">10.1007/s00018-006-6295-0</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2006 </h3>
     <ul>
     
     
    <li>
      <i><b> The transcriptional regulatory code of eukaryotic cells--insights from genome-wide analysis of chromatin organization and transcription factor binding. </b> </i>
     <br>  Leah O. Barrera,  Bing Ren <br>
      <b>  Current opinion in cell biology</b>, 2006;doi:<a href="https://doi.org/10.1016/j.ceb.2006.04.002">10.1016/j.ceb.2006.04.002</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Genome-wide location analysis: insights on transcriptional regulation. </b> </i>
     <br>  R. David Hawkins,  Bing Ren <br>
      <b>  Human molecular genetics</b>, 2006;doi:<a href="https://doi.org/10.1093/hmg/ddl043">10.1093/hmg/ddl043</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Genome-wide analysis of protein-DNA interactions. </b> </i>
     <br>  Tae Hoon Kim,  Bing Ren <br>
      <b>  Annual review of genomics and human genetics</b>, 2006;doi:<a href="https://doi.org/10.1146/annurev.genom.7.080505.115634">10.1146/annurev.genom.7.080505.115634</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> An all-round view of eukaryotic transcription. </b> </i>
     <br>  Tae Hoon Kim,  Bing Ren <br>
      <b>  Genome biology</b>, 2006;doi:<a href="https://doi.org/10.1186/gb-2006-7-7-323">10.1186/gb-2006-7-7-323</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2005 </h3>
     <ul>
     
     
    <li>
      <i><b> A high-resolution map of active promoters in the human genome. </b> </i>
     <br>  Tae Hoon Kim*, Leah O. Barrera*, Ming Zheng, Chunxu Qu, Michael A. Singer, Todd A. Richmond, Yingnian Wu, Roland D. Green,  Bing Ren <br>
      <b>  Nature</b>, 2005;doi:<a href="https://doi.org/10.1038/nature03877">10.1038/nature03877</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Direct isolation and identification of promoters in the human genome. </b> </i>
     <br>  Tae Hoon Kim, Leah O. Barrera, Chunxu Qu, Sara Van Calcar, Nathan D. Trinklein, Sara J. Cooper, Rosa M. Luna, Christopher K. Glass, Michael G. Rosenfeld, Richard M. Myers,  Bing Ren <br>
      <b>  Genome research</b>, 2005;doi:<a href="https://doi.org/10.1101/gr.3430605">10.1101/gr.3430605</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> beta-Catenin activates the growth factor endothelin-1 in colon cancer cells. </b> </i>
     <br>  Tae Hoon Kim, Hui Xiong, Zhuohua Zhang<sup>#</sup>,  Bing Ren<sup>#</sup> <br>
      <b>  Oncogene</b>, 2005;doi:<a href="https://doi.org/10.1038/sj.onc.1208237">10.1038/sj.onc.1208237</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2004 </h3>
     <ul>
     
     
    <li>
      <i><b> The ENCODE (ENCyclopedia Of DNA Elements) Project </b> </i>
     <br>   The ENCODE Project Consortium <br>
      <b>  Science</b>, 2004;doi:<a href="https://doi.org/10.1126/science.1105136">10.1126/science.1105136</a>
    </li>
    <br>
  
  
  
    <li>
      <i><b> Use of chromatin immunoprecipitation assays in genome-wide location analysis of mammalian transcription factors. </b> </i>
     <br>  Bing Ren,  Brian D. Dynlacht <br>
      <b>  Methods in enzymology</b>, 2004;doi:<a href="https://doi.org/10.1016/S0076-6879(03)76020-0">10.1016/S0076-6879(03)76020-0</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2003 </h3>
     <ul>
     
     
    <li>
      <i><b> A global transcriptional regulatory role for c-Myc in Burkitt's lymphoma cells. </b> </i>
     <br>  Zirong Li, Sara Van Calcar, Chunxu Qu, Webster K. Cavenee, Michael Q. Zhang,  Bing Ren <br>
      <b>  Proceedings of the National Academy of Sciences of the United States of America</b>, 2003;doi:<a href="https://doi.org/10.1073/pnas.1332764100">10.1073/pnas.1332764100</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2002 </h3>
     <ul>
     
     
    <li>
      <i><b> E2F integrates cell cycle progression with DNA repair replication and G(2)/M checkpoints. </b> </i>
     <br>  Bing Ren*, Hieu Cam*, Yasuhiko Takahashi, Thomas Volkert, Jolyon Terragni, Richard A. Young,  Brian David Dynlacht <br>
      <b>  Genes & development</b>, 2002;doi:<a href="https://doi.org/10.1101/gad.949802">10.1101/gad.949802</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2001 </h3>
     <ul>
     
     
    <li>
      <i><b> Remodeling of Yeast Genome Expression in Response to Environmental Changes </b> </i>
     <br>  Helen C. Causton*, Bing Ren*, Sang Seok Koh, Christopher T. Harbison, Elenita Kanin, Ezra G. Jennings, Tong Ihn Lee, Heather L. True, Eric S. Lander,  Richard A. Young <br>
      <b>  Molecular Biology of the Cell</b>, 2001;doi:<a href="https://doi.org/10.1091/mbc.12.2.323">10.1091/mbc.12.2.323</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 2000 </h3>
     <ul>
     
     
    <li>
      <i><b> Genome-wide location and function of DNA binding proteins </b> </i>
     <br>  B. Ren*, F. Robert*, J. J. Wyrick*, O. Aparicio, E. G. Jennings, I. Simon, J. Zeitlinger, J. Schreiber, N. Hannett, E. Kanin, T. L. Volkert, C. J. Wilson, S. P. Bell,  R. A. Young <br>
      <b>  Science (New York N.Y.)</b>, 2000;doi:<a href="https://doi.org/10.1126/science.290.5500.2306">10.1126/science.290.5500.2306</a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 1999 </h3>
     <ul>
     
     
    <li>
      <i><b> PRDI-BF1/Blimp-1 repression is mediated by corepressors of the Groucho family of proteins </b> </i>
     <br>  Bing Ren, Kerlen J. Chee, Tae Hoon Kim,  Tom Maniatis <br>
      <b>  Genes & Development</b>, 1999;doi:<a href="https://doi.org/"></a>
    </li>
    <br>
  
  
  
   </ul>
     <h3> 1998 </h3>
     <ul>
     
     
    <li>
      <i><b> Regulation of Drosophila Adh promoter switching by an initiator-targeted repression mechanism. </b> </i>
     <br>  B Ren,  T Maniatis <br>
      <b>  The EMBO Journal</b>, 1998;doi:<a href="https://doi.org/10.1093/emboj/17.4.1076">10.1093/emboj/17.4.1076</a>
    </li>
    <br>
  
</ul>


			</div>



			
		</article>
	</div><!-- /.medium-8.columns -->


	


	
</div><!-- /.row -->


	
	     <div id="up-to-top" class="row">
      <div class="small-12 columns" style="text-align: right;">
        <a class="iconfont" href="#top-of-page">&#xf108;</a>
      </div><!-- /.small-12.columns -->
    </div><!-- /.row -->


    <footer id="footer-content" class="bg-grau">

      <div id="subfooter">
        <nav class="row">
          <section id="subfooter-left" class="small-12 medium-6 columns credits">
            <p>Created with &hearts; by <a href="https://shawnzhangyx.github.io/">Yanxiao Zhang</a> with <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> based on <a href="http://phlow.github.io/feeling-responsive/">Feeling Responsive</a>.</p>
          </section>

          <section id="subfooter-right" class="small-12 medium-6 columns">
            <ul class="inline-list social-icons">
            
              <li><a href="http://github.com/ren-lab" target="_blank" class="icon-github" title="Code und mehr..."></a></li>
            
            </ul>
          </section>
        </nav>
      </div><!-- /#subfooter -->
    </footer>

	

	


<script src="/assets/js/javascript.min.js"></script>







<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-96143943-2', 'auto');
  ga('set', 'anonymizeIp', true);
  ga('send', 'pageview');

</script>








</body>
</html>

