<!doctype html>
<html class="no-js" lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>ENCODE3 mouse histone and ATAC-seq paper supplementary data files</title>

    <link rel="stylesheet" type="text/css" href="/assets/css/styles_feeling_responsive.css">

  

	<script src="/assets/js/modernizr.min.js"></script>

	<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
	<script>
		WebFont.load({
			google: {
				families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
			}
		});
	</script>

	<noscript>
		<link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic%7CVolkhov' rel='stylesheet' type='text/css'>
	</noscript>


	<!-- Search Engine Optimization -->
	<meta name="description" content="These files are supplementary files for the ENCODE3 mouse chromatin state and accessibility paper (Gorkin, Barozzi, Zhao, Zhang, Huang et al. 2020). They include the following files:


  Raw and processed ChIP-seq data from our study can be accessed via the ENCODE Data Collection and Coordination (DCC) website: http://www.encodedcc.org.  Here is a list of accession IDs for all samples.
  Publication landing page at the ENCODE DCC portal, which contains links to relevant datasets.
  chromHMM calls
  
    H3K27ac dynamic enhancer calls
  
  ATAC-seq peak calls and signal tracks
  ATAC-seq dynamic d-TAC calls
  dTAC catalog accessibility by tissue-stage


Note: Processed ATAC-seq files used in the paper may be different from those available via the ENCODE DCC portal, because the ENCODE DCC did not have a official processing pipeline for ATAC-seq at the time of data analysis. However, the raw data used here and by the ENCODE DCC portal are identical. For most purposes, we recommend using the processed data files available on the ENCODE DCC portal. The files here are made available to allow precise reproduction of results from the paper.">
	<meta name="google-site-verification" content="Vk0IOJ2jwG_qEoG7fuEXYqv0m2rLa8P778Fi_GrsgEQ">
	<meta name="msvalidate.01" content="0FB4C028ABCF07C908C54386ABD2D97F" >
	
	<link rel="author" href="https://plus.google.com/u/0/118311555303973066167">
	
	
	<link rel="canonical" href="/download/encode3-mouse-histone-atac/">


	<!-- Facebook Open Graph -->
	<meta property="og:title" content="ENCODE3 mouse histone and ATAC-seq paper supplementary data files">
	<meta property="og:description" content="These files are supplementary files for the ENCODE3 mouse chromatin state and accessibility paper (Gorkin, Barozzi, Zhao, Zhang, Huang et al. 2020). They include the following files:


  Raw and processed ChIP-seq data from our study can be accessed via the ENCODE Data Collection and Coordination (DCC) website: http://www.encodedcc.org.  Here is a list of accession IDs for all samples.
  Publication landing page at the ENCODE DCC portal, which contains links to relevant datasets.
  chromHMM calls
  
    H3K27ac dynamic enhancer calls
  
  ATAC-seq peak calls and signal tracks
  ATAC-seq dynamic d-TAC calls
  dTAC catalog accessibility by tissue-stage


Note: Processed ATAC-seq files used in the paper may be different from those available via the ENCODE DCC portal, because the ENCODE DCC did not have a official processing pipeline for ATAC-seq at the time of data analysis. However, the raw data used here and by the ENCODE DCC portal are identical. For most purposes, we recommend using the processed data files available on the ENCODE DCC portal. The files here are made available to allow precise reproduction of results from the paper.">
	<meta property="og:url" content="//download/encode3-mouse-histone-atac/">
	<meta property="og:locale" content="en_EN">
	<meta property="og:type" content="website">
	<meta property="og:site_name" content="RENLAB">
	
	<meta property="article:author" content="https://www.facebook.com/phlow.media">


	
	<!-- Twitter -->
	<meta name="twitter:card" content="summary">
	<meta name="twitter:site" content="phlow">
	<meta name="twitter:creator" content="phlow">
	<meta name="twitter:title" content="ENCODE3 mouse histone and ATAC-seq paper supplementary data files">
	<meta name="twitter:description" content="These files are supplementary files for the ENCODE3 mouse chromatin state and accessibility paper (Gorkin, Barozzi, Zhao, Zhang, Huang et al. 2020). They include the following files:


  Raw and processed ChIP-seq data from our study can be accessed via the ENCODE Data Collection and Coordination (DCC) website: http://www.encodedcc.org.  Here is a list of accession IDs for all samples.
  Publication landing page at the ENCODE DCC portal, which contains links to relevant datasets.
  chromHMM calls
  
    H3K27ac dynamic enhancer calls
  
  ATAC-seq peak calls and signal tracks
  ATAC-seq dynamic d-TAC calls
  dTAC catalog accessibility by tissue-stage


Note: Processed ATAC-seq files used in the paper may be different from those available via the ENCODE DCC portal, because the ENCODE DCC did not have a official processing pipeline for ATAC-seq at the time of data analysis. However, the raw data used here and by the ENCODE DCC portal are identical. For most purposes, we recommend using the processed data files available on the ENCODE DCC portal. The files here are made available to allow precise reproduction of results from the paper.">
	
	

	<link type="text/plain" rel="author" href="/humans.txt">

	

	

	<link rel="icon" sizes="32x32" href="/assets/img/favicon-32x32.png">

	<link rel="icon" sizes="192x192" href="/assets/img/touch-icon-192x192.png">

	<link rel="apple-touch-icon-precomposed" sizes="180x180" href="/assets/img/apple-touch-icon-180x180-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="152x152" href="/assets/img/apple-touch-icon-152x152-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="144x144" href="/assets/img/apple-touch-icon-144x144-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="120x120" href="/assets/img/apple-touch-icon-120x120-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="114x114" href="/assets/img/apple-touch-icon-114x114-precomposed.png">

	
	<link rel="apple-touch-icon-precomposed" sizes="76x76" href="/assets/img/apple-touch-icon-76x76-precomposed.png">

	<link rel="apple-touch-icon-precomposed" sizes="72x72" href="/assets/img/apple-touch-icon-72x72-precomposed.png">

	<link rel="apple-touch-icon-precomposed" href="/assets/img/apple-touch-icon-precomposed.png">	

	<meta name="msapplication-TileImage" content="//assets/img/msapplication_tileimage.png">

	<meta name="msapplication-TileColor" content="#fabb00">


	

</head>
<body id="top-of-page" class="post">
	
	<div id="navigation" class="sticky">
  <nav class="top-bar" role="navigation" data-topbar>
    <ul class="title-area">
      <li class="name">
      <h1 class="show-for-small-only"><a href="/" class="icon-tree"> RENLAB</a></h1>
    </li>
       <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->
      <li class="toggle-topbar menu-icon"><a href="#"><span>Nav</span></a></li>
    </ul>
    <section class="top-bar-section">

      <ul class="right">
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
            
            
              <li class="divider"></li>
              <li><a  href="/search/">Search</a></li>

            
            
          
        

              

          
          
            
            
              <li class="divider"></li>
              <li><a  href="/contact/">Contact</a></li>

            
            
          
        
        
      </ul>

      <ul class="left">
        

              

          
          

            
            
              <li><a  href="/">Home</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="/news/">News</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="/research/">Research</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            
              <li><a  href="/team/">Team</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="/publication/">Publication</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="/publication/preprint/">Preprint</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          

            
            
              <li><a  href="/download/">Download</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          
        

              

          
          
        
        
      </ul>
    </section>
  </nav>
</div><!-- /#navigation -->

	

	

<div id="masthead-no-image-header">
	<div class="row">
		<div class="small-12 columns">
			<a id="logo" href="/" title="RENLAB – Laboratory of Gene Regulation">
				<img src="/assets/img/logo.png" alt="RENLAB – Laboratory of Gene Regulation">
			</a>
		</div><!-- /.small-12.columns -->
	</div><!-- /.row -->
</div><!-- /#masthead -->




<nav class="breadcrumbs" role="menubar" aria-label="breadcrumbs">
 <a href="/">Start</a>
 
   
    
        <a href="/download/">download</a>
    
  
    
        <a class="current">ENCODE3 mouse histone and ATAC-seq paper supplementary data files</a>
    
  
</nav>









	<div class="row t30">
	<div class="medium-8 columns medium-offset-2 end">
		<article itemscope itemtype="http://schema.org/Article">
			<header>
				

				<div itemprop="name">
					<p class="subheadline">Data/download</p>
					<h1>ENCODE3 mouse histone and ATAC-seq paper supplementary data files</h1>
				</div>
			</header>


			

			<div itemprop="articleSection">
			<p>These files are supplementary files for the ENCODE3 mouse chromatin state and accessibility paper (Gorkin, Barozzi, Zhao, Zhang, Huang et al. 2020). They include the following files:</p>

<ul>
  <li>Raw and processed ChIP-seq data from our study can be accessed via the ENCODE Data Collection and Coordination (DCC) website: <a href="http://www.encodedcc.org">http://www.encodedcc.org</a>.  Here is a <a href="/renlab_website/download_files/ENCODE3_mouse_tableS13.xlsx">list</a> of accession IDs for all samples.</li>
  <li><a href="https://www.encodeproject.org/publications/8468259b-3be2-4483-970c-9ce1d4d93f3a/">Publication landing page at the ENCODE DCC portal</a>, which contains links to relevant datasets.</li>
  <li><a href="http://enhancer.sdsc.edu/enhancer_export/ENCODE/chromHMM/">chromHMM calls</a></li>
  <li>
    <p><a href="/renlab_website/download_files/ENCODE3_mouse_dynamic_H3K27ac_peaks.txt.gz">H3K27ac dynamic enhancer calls</a></p>
  </li>
  <li><a href="http://renlab.sdsc.edu/yanxiao/encode_trackhub/mm10/atacseq/">ATAC-seq peak calls and signal tracks</a></li>
  <li><a href="/renlab_website/download_files/ENCODE3_mouse_dynamic_dTACs.txt.gz">ATAC-seq dynamic d-TAC calls</a></li>
  <li><a href="/renlab_website/download_files/dTAC_catalog_accessibility_by_sample.tar.gz">dTAC catalog accessibility by tissue-stage</a></li>
</ul>

<p>Note: Processed ATAC-seq files used in the paper may be different from those available via the ENCODE DCC portal, because the ENCODE DCC did not have a official processing pipeline for ATAC-seq at the time of data analysis. However, the raw data used here and by the ENCODE DCC portal are identical. For most purposes, we recommend using the processed data files available on the ENCODE DCC portal. The files here are made available to allow precise reproduction of results from the paper.</p>

			</div>



			
		</article>
	</div><!-- /.medium-8.columns -->


	


	
</div><!-- /.row -->


	
	     <div id="up-to-top" class="row">
      <div class="small-12 columns" style="text-align: right;">
        <a class="iconfont" href="#top-of-page">&#xf108;</a>
      </div><!-- /.small-12.columns -->
    </div><!-- /.row -->


    <footer id="footer-content" class="bg-grau">

      <div id="subfooter">
        <nav class="row">
          <section id="subfooter-left" class="small-12 medium-6 columns credits">
            <p>Created with &hearts; by <a href="https://shawnzhangyx.github.io/">Yanxiao Zhang</a> with <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> based on <a href="http://phlow.github.io/feeling-responsive/">Feeling Responsive</a>.</p>
          </section>

          <section id="subfooter-right" class="small-12 medium-6 columns">
            <ul class="inline-list social-icons">
            
              <li><a href="http://github.com/ren-lab" target="_blank" class="icon-github" title="Code und mehr..."></a></li>
            
            </ul>
          </section>
        </nav>
      </div><!-- /#subfooter -->
    </footer>

	

	


<script src="/assets/js/javascript.min.js"></script>







<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-96143943-2', 'auto');
  ga('set', 'anonymizeIp', true);
  ga('send', 'pageview');

</script>








</body>
</html>

